Description: use the system copy of elib.intl
Author: Jonathan Wiltshire <jmw@debian.org>
Forwarded: no
Last-Update: 2011-08-09

Index: rednotebook-1.10.3/rednotebook/journal.py
===================================================================
--- rednotebook-1.10.3.orig/rednotebook/journal.py	2015-10-15 20:20:46.118409462 +0200
+++ rednotebook-1.10.3/rednotebook/journal.py	2015-10-15 20:20:46.114409318 +0200
@@ -45,7 +45,7 @@
 
 # ---------------------- Enable i18n -------------------------------
 
-from rednotebook.external import elibintl
+import elib.intl
 
 # We need to translate 3 different types of strings:
 # * sourcecode strings
@@ -59,7 +59,7 @@
 
 # Register _() as a global translation function and set up the translation
 try:
-    elibintl.install(GETTEXT_DOMAIN, LOCALE_PATH)
+    elib.intl.install(GETTEXT_DOMAIN, LOCALE_PATH)
 except locale.Error, err:
     # unsupported locale setting
     logging.error('Locale could not be set: "%s"' % err)
