# remove invalid category
Index: meteo-0.9.4+git161129/share/meteo-qt.desktop
===================================================================
--- meteo-0.9.4+git161129.orig/share/meteo-qt.desktop	2016-11-29 22:07:41.245033779 +0100
+++ meteo-0.9.4+git161129/share/meteo-qt.desktop	2016-11-29 22:07:50.253222892 +0100
@@ -24,5 +24,5 @@
 Icon=meteo-qt
 Terminal=false
 StartupNotify=false
-Categories=Qt;LXQt;Network;
+Categories=Qt;Network;
 X-DBUS-ServiceName=meteo-qt
