Index: avidemux2.6-2.6.12/bootStrap.bash
===================================================================
--- avidemux2.6-2.6.12.orig/bootStrap.bash	2016-03-05 21:16:51.005983373 +0100
+++ avidemux2.6-2.6.12/bootStrap.bash	2016-03-05 21:16:50.997983210 +0100
@@ -183,7 +183,7 @@
 
 export TOP=$PWD
 export POSTFIX=""
-export FAKEROOT_DIR=$PWD/install
+export FAKEROOT_DIR=$PWD/debian/tmp
 echo "Top dir : $TOP"
 echo "Fake installation directory=$FAKEROOT_DIR"
 if [ "x$debug" = "x1" ] ; then echo   
