Index: avidemux2.6-2.6.10/bootStrap.bash
===================================================================
--- avidemux2.6-2.6.10.orig/bootStrap.bash	2015-07-08 22:24:12.219386699 +0200
+++ avidemux2.6-2.6.10/bootStrap.bash	2015-07-08 22:24:12.215386601 +0200
@@ -171,7 +171,7 @@
 
 export TOP=$PWD
 export POSTFIX=""
-export FAKEROOT_DIR=$PWD/install
+export FAKEROOT_DIR=$PWD/debian/tmp
 echo "Top dir : $TOP"
 echo "Fake installation directory=$FAKEROOT_DIR"
 if [ "x$debug" = "x1" ] ; then echo   
