Index: avidemux2.6-2.6.4/bootStrap.bash
===================================================================
--- avidemux2.6-2.6.4.orig/bootStrap.bash	2013-05-09 13:10:35.000000000 +0200
+++ avidemux2.6-2.6.4/bootStrap.bash	2013-05-10 19:36:04.180011685 +0200
@@ -164,7 +164,7 @@
 
 export TOP=$PWD
 export POSTFIX=""
-export FAKEROOT_DIR=$PWD/install
+export FAKEROOT_DIR=$PWD/debian/tmp
 echo "Top dir : $TOP"
 echo "Fake installation directory=$FAKEROOT_DIR"
 if [ "x$debug" = "x1" ] ; then echo   
