Description: Look for JAR in /usr/lib/ps3mediaserver
Index: ps3mediaserver-1.90.1/src/main/external-resources/PMS.sh
===================================================================
--- ps3mediaserver-1.90.1.orig/src/main/external-resources/PMS.sh	2014-08-02 21:34:01.806061916 +0200
+++ ps3mediaserver-1.90.1/src/main/external-resources/PMS.sh	2014-08-02 21:35:41.582058948 +0200
@@ -1,23 +1,6 @@
 #!/bin/sh
 
-CMD=`readlink -f $0`
-DIRNAME=`dirname "$CMD"`
-
-# OS specific support (must be 'true' or 'false').
-cygwin=false;
-case "`uname`" in
-    CYGWIN*)
-        cygwin=true
-        ;;
-esac
-
-# For Cygwin, ensure paths are in UNIX format before anything is touched
-if $cygwin ; then
-    [ -n "$PMS_HOME" ] &&
-		PMS_HOME=`cygpath --unix "$PMS_HOME"`
-    [ -n "$JAVA_HOME" ] &&
-		JAVA_HOME=`cygpath --unix "$JAVA_HOME"`
-fi
+DIRNAME=/usr/lib/ps3mediaserver
 
 # Setup PMS_HOME
 if [ "x$PMS_HOME" = "x" ]; then
@@ -37,10 +20,15 @@
     fi
 fi
 
+# Setup the JVM mem usage
+if [ "x$PS3MS_MEM" = "x" ]; then
+	PS3MS_MEM="768M"
+fi
+
 # Setup the classpath
 # since we always cd to the working dir, these a) can be unqualified and b) *must*
 # be unqualified: https://code.google.com/p/ps3mediaserver/issues/detail?id=1122
-PMS_JARS="update.jar:pms.jar"
+PMS_JARS="pms.jar"
 
 # For Cygwin, switch paths to Windows format before running java
 if $cygwin; then
@@ -59,4 +47,4 @@
 fi
 
 # Execute the JVM
-exec "$JAVA" $JAVA_OPTS -Xmx768M -Xss1024k -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djna.nosys=true -classpath "$PMS_JARS" net.pms.PMS "$@"
+exec "$JAVA" $JAVA_OPTS -Xmx$PS3MS_MEM -Xss1024k -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djna.nosys=true -classpath "$PMS_JARS" net.pms.PMS "$@"
