Index: ps3mediaserver-1.90.1/src/main/external-resources/PMS.conf
===================================================================
--- ps3mediaserver-1.90.1.orig/src/main/external-resources/PMS.conf	2014-08-02 21:20:54.270085343 +0200
+++ ps3mediaserver-1.90.1/src/main/external-resources/PMS.conf	2014-08-02 21:29:06.358070704 +0200
@@ -168,7 +168,7 @@
 # If your host server has a power-saving mode, use this option to prevent the
 # server hibernating or sleeping while it is streaming data.
 # Default: false
-prevents_sleep_mode =
+prevents_sleep_mode = true
 
 # GUI Option: Default renderer when automatic detection fails (selector)
 # PMS can recognize many renderers automatically. Sometimes a renderer will
@@ -243,7 +243,7 @@
 # Whether or not to use MPlayer to generate the thumbnails for video; can
 # produce better results than built-in method.
 # Default: false, which will use the built-in method.
-use_mplayer_for_video_thumbs =
+use_mplayer_for_video_thumbs = true
 
 # DVD thumbnails
 # --------------
@@ -416,7 +416,7 @@
 # Warning: the value can be empty, which overrides the default value and will
 # disable all engines.
 # Default: "mencoder,avsmencoder,tsmuxer,ffmpegvideo,vlctranscoder,ffmpegaudio,tsmuxeraudio,ffmpegwebvideo,vlcvideo,mencoderwebvideo,vlcwebvideo,vlcaudio,ffmpegdvrmsremux,rawthumbs"
-# engines =
+engines = tsmuxer,mencoder,ffmpegvideo,vlctranscoder,ffmpegaudio,mplayeraudio,tsmuxeraudio,ffmpegwebvideo,vlcvideo,mencoderwebvideo,mplayervideodump,mplayerwebaudio,vlcaudio,rawthumbs
 
 # ----------------------------------------------------------------------------
 # Transcoding Settings Tab: Common Transcode Settings Page
@@ -476,7 +476,7 @@
 # Sets the quality for MPEG-2 conversion; MPEG-2 is the fallback format if the
 # client does not support a video format natively.
 # Default: "Automatic (Wired)"
-mpeg2_main_settings =
+mpeg2_main_settings = keyint=2:vqscale=1:vqmin=2
 
 # Skip transcode for extensions
 # -----------------------------
@@ -534,7 +534,7 @@
 # GUI Option: Keep DTS tracks (no re-encode, compatible with AV receiver
 # through optical or HDMI output) (toggle)
 # Default: false
-audio_embed_dts_in_pcm =
+audio_embed_dts_in_pcm = true
 
 # AC3 transcoding audio bitrate
 # -----------------
@@ -702,7 +702,7 @@
 # GUI Option: Subtitle font (string)
 # GUI offers a navigator to a folder generating an absolute path.
 # Default: ""
-subtitle_font =
+subtitle_font = Ubuntu
 
 # Use ASS/SSA styling
 # -------------------
@@ -730,7 +730,7 @@
 # from that will generate a number. Default is empty, causing PMS to pick
 # white.
 # Default: ""
-subtitle_color =
+subtitle_color = -1
 
 # Style font scale
 # ----------------
@@ -742,17 +742,17 @@
 # GUI Option: Styled font outline
 # The outline parameter used for ASS subtitling.
 # Default: 1
-subtitle_ass_outline =
+subtitle_ass_outline = 2
 
 # GUI Option: Styled font shadow
 # The shadow parameter used for ASS subtitling.
 # Default: 1
-subtitle_ass_shadow =
+subtitle_ass_shadow = 0
 
 # GUI Option: Styled margin
 # The margin used for ASS subtitling.
 # Default: 10
-subtitle_ass_margin =
+subtitle_ass_margin = 40
 
 # Plain font scale
 # ----------------
@@ -764,7 +764,7 @@
 # GUI Option: Plain font outline
 # The subfont outline parameter used for subtitling without ASS.
 # Default: 1
-mencoder_noass_outline =
+mencoder_noass_outline = 4
 
 # GUI Option: Plain font blur
 # The subfont blur parameter used for subtitling without ASS.
@@ -774,13 +774,13 @@
 # GUI Option: Plain margin
 # The subpos parameter used for subtitling without ASS.
 # Default: 2
-mencoder_noass_subpos =
+mencoder_noass_subpos = 13
 
 # DVD/VOB subtitle quality
 # ------------------------
 # GUI Option: DVD/VOB subtitle quality
 # Default: 3
-mencoder_vobsub_subtitle_quality =
+mencoder_vobsub_subtitle_quality = 4
 
 # Add borders
 # -----------
@@ -866,35 +866,35 @@
 #     Win: win32/mencoder.exe
 #     Mac: osx/mencoder
 #     Linux: mencoder + system PATH
-mencoder_path =
+mencoder_path = /usr/lib/ps3mediaserver/linux/mencoder
 
 # Path to ffmpeg (absolute or relative from project.binaries.dir)
 # Default:
 #     Win: win32/ffmpeg.exe
 #     Mac: osx/ffmpeg
 #     Linux: ffmpeg + system PATH
-ffmpeg_path =
+ffmpeg_path = /usr/lib/ps3mediaserver/linux/ffmpeg
 
 # Path to mplayer (absolute or relative from project.binaries.dir)
 # Default:
 #     Win: win32/mplayer.exe
 #     Mac: osx/mplayer
 #     Linux: mplayer + system PATH
-mplayer_path =
+mplayer_path = /usr/lib/ps3mediaserver/linux/mplayer
 
 # Path to tsMuxeR (absolute or relative from project.binaries.dir)
 # Default:
 #     Win: win32/tsMuxeR.exe
 #     Mac: osx/tsMuxeR
 #     Linux: tsMuxeR + system PATH
-tsmuxer_path =
+tsmuxer_path = /usr/lib/ps3mediaserver/linux/tsMuxeR
 
 # Path to dcraw (absolute or relative from project.binaries.dir)
 # Default:
 #     Win: win32/dcrawMS.exe
 #     Mac: osx/dcraw
 #     Linux: dcraw + system PATH
-dcraw_path =
+dcraw_path = /usr/lib/ps3mediaserver/linux/dcraw
 
 # Path to FLAC (absolute or relative from project.binaries.dir)
 # Default:
