Description: Remove unstable warning in window title
Index: ps3mediaserver-1.90.1/src/main/java/net/pms/newgui/LooksFrame.java
===================================================================
--- ps3mediaserver-1.90.1.orig/src/main/java/net/pms/newgui/LooksFrame.java	2014-08-02 21:33:42.690062484 +0200
+++ ps3mediaserver-1.90.1/src/main/java/net/pms/newgui/LooksFrame.java	2014-08-02 21:33:42.686062484 +0200
@@ -265,11 +265,6 @@
 		String projectVersion = PropertiesUtil.getProjectProperties().get("project.version");
 		String title = projectName + " " + projectVersion;
 
-		// If the version contains a "-" (e.g. "1.50.1-SNAPSHOT" or "1.50.1-beta1"), add a warning message
-		if (projectVersion.indexOf("-") > -1) {
-			title = title + " - " + Messages.getString("LooksFrame.26");
-		}
-
 		this.setTitle(title);
 		this.setDefaultCloseOperation(JFrame.HIDE_ON_CLOSE);
 		Dimension screenSize = getToolkit().getScreenSize();
