# update versions
Index: ps3mediaserver-1.90.1/contrib/binaries-deps-versions
===================================================================
--- ps3mediaserver-1.90.1.orig/contrib/binaries-deps-versions	2013-08-17 16:52:16.000000000 +0200
+++ ps3mediaserver-1.90.1/contrib/binaries-deps-versions	2014-08-02 22:44:21.845936381 +0200
@@ -9,7 +9,7 @@
 #    mplayer-configure.patch             MPlayer r35003
 #
 VERSION_BZIP2=1.0.6
-VERSION_DCRAW=9.07
+VERSION_DCRAW=9.22
 VERSION_ENCA=1.13
 VERSION_EXPAT=2.1.0
 VERSION_FAAD2=2.7
