Do not check for updates.
Index: rbutil-1.3.1/rbutil/rbutilqt/rbutilqt.cpp
===================================================================
--- rbutil-1.3.1.orig/rbutil/rbutilqt/rbutilqt.cpp	2012-11-25 14:25:19.000000000 +0100
+++ rbutil-1.3.1/rbutil/rbutilqt/rbutilqt.cpp	2012-12-01 17:03:23.654579928 +0100
@@ -654,7 +654,7 @@
 
 void RbUtilQt::downloadUpdateDone(bool error)
 {
-    if(error) {
+/*    if(error) {
         qDebug() << "[RbUtil] network error:" << update->error();
     }
     else {
@@ -722,7 +722,7 @@
         else {
             ui.statusbar->showMessage(tr("Rockbox Utility is up to date."), 5000);
         }
-    }
+    }*/
 }
 
 
