Do not check for updates.
Index: rbutil-1.4.0/rbutil/rbutilqt/rbutilqt.cpp
===================================================================
--- rbutil-1.4.0.orig/rbutil/rbutilqt/rbutilqt.cpp	2014-01-10 21:35:06.649508525 +0100
+++ rbutil-1.4.0/rbutil/rbutilqt/rbutilqt.cpp	2014-01-10 21:35:57.293507019 +0100
@@ -667,7 +667,7 @@
 
 void RbUtilQt::downloadUpdateDone(bool error)
 {
-    if(error) {
+    /*if(error) {
         LOG_INFO() << "network error:" << update->errorString();
     }
     else {
@@ -735,7 +735,7 @@
         else {
             ui.statusbar->showMessage(tr("Rockbox Utility is up to date."), 5000);
         }
-    }
+    }*/
 }
 
 
