https://github.com/sahib/glyr/pull/16
Index: libglyr-0.0.0+git20120214/CMakeLists.txt
===================================================================
--- libglyr-0.0.0+git20120214.orig/CMakeLists.txt	2012-02-16 21:33:40.835667281 +0100
+++ libglyr-0.0.0+git20120214/CMakeLists.txt	2012-02-16 21:34:04.195666586 +0100
@@ -57,7 +57,7 @@
 MESSAGE("-- Building Target: ${CMAKE_BUILD_TYPE}")
 
 SET(COMMON_FLAGS "-Wall -Wextra -Wstrict-prototypes -std=c99 -W -Wno-unused-parameter -Wno-strict-prototypes")
-SET(CMAKE_C_FLAGS_RELEASE "${COMMON_FLAGS} ${CMAKE_C_FLAGS} -Os -march=native -s")
+SET(CMAKE_C_FLAGS_RELEASE "${COMMON_FLAGS} ${CMAKE_C_FLAGS} -Os -s")
 SET(CMAKE_C_FLAGS_DEBUG   "${COMMON_FLAGS} ${CMAKE_C_FLAGS} -ggdb3")
 SET(CMAKE_C_FLAGS "${COMMON_FLAGS} ${CMAKE_C_FLAGS}")
 # ------------------------------------------------
