# hardcode default path to ffmpeg
Index: slowmovideo-0.5/src/slowmoVideo/slowmoUI/mainwindow.cpp
===================================================================
--- slowmovideo-0.5.orig/src/slowmoVideo/slowmoUI/mainwindow.cpp	2016-02-06 23:10:21.916944035 +0100
+++ slowmovideo-0.5/src/slowmoVideo/slowmoUI/mainwindow.cpp	2016-02-06 23:10:21.912943949 +0100
@@ -185,10 +185,11 @@
     }
     
     if (!settings.contains("binaries/ffmpeg")) {
-        qDebug() << "need to find ffmpeg";
+        /*qDebug() << "need to find ffmpeg";
 	QMessageBox::information( this, 
 		"valid FFMPEG not found", "Please choose a working ffmpeg\n" , 
-		QMessageBox::Ok, 0 );
+		QMessageBox::Ok, 0 );*/
+        settings.setValue("binaries/ffmpeg", "/usr/bin/ffmpeg");
         PreferencesDialog dialog;
         dialog.exec();
     }
