Description: Replace gnomevfs methods, now deprecated.
Origin: Debian
Bug-Debian: http://bugs.debian.org/605555
Forwarded: https://launchpad.net/bugs/708898

Index: bleachbit-0.8.8/bleachbit/Unix.py
===================================================================
--- bleachbit-0.8.8.orig/bleachbit/Unix.py	2011-01-27 23:18:08.071852091 +0100
+++ bleachbit-0.8.8/bleachbit/Unix.py	2011-01-27 23:20:04.531129734 +0100
@@ -36,18 +36,11 @@
 import FileUtilities
 import General
 
-HAVE_GNOME_VFS = True
+HAVE_GIO = True
 try:
-    import gnomevfs
+    import gio
 except:
-    try:
-        # this is the deprecated name
-        import gnome.vfs
-    except:
-        HAVE_GNOME_VFS = False
-    else:
-        gnomevfs = gnome.vfs
-
+    HAVE_GIO = False
 
 
 def guess_overwrite_paths():
@@ -483,7 +483,7 @@
             print "info: is_broken_xdg_menu: missing required option 'MimeType': '%s'" % (pathname)
             return True
         mimetype = config.get('Desktop Entry', 'MimeType').strip().lower()
-        if HAVE_GNOME_VFS and 0 == len(gnomevfs.mime_get_all_applications(mimetype)):
+        if HAVE_GIO and 0 == len(gio.app_info_get_all_for_type(mimetype)):
             print "info: is_broken_xdg_menu: MimeType '%s' not " \
                 "registered '%s'" % (mimetype, pathname)
             return True
