# fix include path
Index: beebeep-3.0.6/src/gui/AudioManager.h
===================================================================
--- beebeep-3.0.6.orig/src/gui/AudioManager.h	2016-03-21 20:32:34.000000000 +0100
+++ beebeep-3.0.6/src/gui/AudioManager.h	2016-04-28 22:30:00.151470174 +0200
@@ -26,7 +26,7 @@
 
 #include "Config.h"
 #ifdef BEEBEEP_USE_PHONON4
-  #include <Phonon/MediaObject>
+  #include <KDE/Phonon/MediaObject>
 #endif
 
 
