diff -Nur -x '*.orig' -x '*~' gimp-2.6.1/plug-ins/help/gimphelpdomain.c gimp-2.6.1.new/plug-ins/help/gimphelpdomain.c
--- gimp-2.6.1/plug-ins/help/gimphelpdomain.c	2008-08-21 13:35:16.000000000 +0200
+++ gimp-2.6.1.new/plug-ins/help/gimphelpdomain.c	2008-10-06 13:35:46.000000000 +0200
@@ -185,9 +185,9 @@
                 {
                   g_message ("%s\n\n%s",
                              _("The GIMP user manual is not available."),
-                             _("Please install the additional help package "
-                               "or use the online user manual at "
-                               "http://docs.gimp.org/."));
+                             _("Please make sure gimp-help-en is installed, or "
+                               "the appropriate gimp-help package for your "
+                               "language."));
                 }
               break;
 
