# 
# Description: change the menu item label
#
diff -Nur -x '*.orig' -x '*~' gimp-2.6.1/desktop/gimp.desktop.in.in gimp-2.6.1.new/desktop/gimp.desktop.in.in
--- gimp-2.6.1/desktop/gimp.desktop.in.in	2008-08-21 13:34:07.000000000 +0200
+++ gimp-2.6.1.new/desktop/gimp.desktop.in.in	2008-10-06 13:39:59.000000000 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-_Name=GNU Image Manipulation Program
+_Name=GIMP Image Editor
 _GenericName=Image Editor
 _Comment=Create images and edit photographs
 Exec=@GIMP_COMMAND@ %U

