# make rssguard compatible with qt5.5
Index: rssguard-3.4.1/src/main.cpp
===================================================================
--- rssguard-3.4.1.orig/src/main.cpp	2017-07-15 16:19:49.914437448 +0200
+++ rssguard-3.4.1/src/main.cpp	2017-07-15 16:20:39.733686740 +0200
@@ -83,7 +83,7 @@
   application.setFeedReader(new FeedReader(&application));
 
   QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
-  QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+  //QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #ifdef Q_OS_MAC
   QApplication::setAttribute(Qt::AA_DontShowIconsInMenus);
   disableWindowTabbing();
