# make_unique is a new feature
Index: freefilesync-9.1/Makefile
===================================================================
--- freefilesync-9.1.orig/Makefile	2017-05-24 17:22:00.417898138 +0200
+++ freefilesync-9.1/Makefile	2017-05-24 17:22:00.417898138 +0200
@@ -5,7 +5,7 @@
 APPSHAREDIR = $(SHAREDIR)/$(APPNAME)
 DOCSHAREDIR = $(SHAREDIR)/doc/$(APPNAME)
 
-CXXFLAGS  = -std=c++14 -pipe -DWXINTL_NO_GETTEXT_MACRO -I. -I./zenXml -include "zen/i18n.h" -Wall -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
+CXXFLAGS  = -std=c++14 -std=gnu++14 -pipe -DWXINTL_NO_GETTEXT_MACRO -I. -I./zenXml -include "zen/i18n.h" -Wall -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
 
 LINKFLAGS = -s `wx-config --libs std, aui --debug=no` -lboost_thread -lboost_chrono -lboost_system -lz -pthread
 
Index: freefilesync-9.1/RealTimeSync/Makefile
===================================================================
--- freefilesync-9.1.orig/RealTimeSync/Makefile	2017-05-24 17:22:00.417898138 +0200
+++ freefilesync-9.1/RealTimeSync/Makefile	2017-05-24 17:22:18.929601528 +0200
@@ -2,7 +2,7 @@
 prefix = /usr
 BINDIR = $(DESTDIR)$(prefix)/bin
 
-CXXFLAGS  = -std=c++14 -pipe -DWXINTL_NO_GETTEXT_MACRO -I.. -I../zenXml -include "zen/i18n.h" -Wall -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
+CXXFLAGS  = -std=c++14 -std=gnu++14 -pipe -DWXINTL_NO_GETTEXT_MACRO -I.. -I../zenXml -include "zen/i18n.h" -Wall -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
 
 LINKFLAGS = -s `wx-config --libs std, aui --debug=no` -lboost_thread -lboost_chrono -lboost_system -lz -pthread
 
