# add missing link
Index: freefilesync-6.7/Makefile
===================================================================
--- freefilesync-6.7.orig/Makefile	2014-07-15 17:57:26.033467340 +0200
+++ freefilesync-6.7/Makefile	2014-07-15 17:57:26.029467340 +0200
@@ -62,7 +62,7 @@
 debug:     LINKFLAGS += `wx-config   --libs std, aui --debug=yes `
 launchpad: LINKFLAGS += `wx-config                              --libs std, aui --debug=no`
 
-release debug: LINKFLAGS += -lboost_thread -lboost_system
+release debug: LINKFLAGS += -lboost_thread -lboost_system -lz
 launchpad:     LINKFLAGS += -lboost_thread -lboost_system -lz
 
 
