# add missing link
Index: freefilesync-5.13/Makefile
===================================================================
--- freefilesync-5.13.orig/Makefile	2014-03-29 15:40:25.000000000 +0100
+++ freefilesync-5.13/Makefile	2014-03-29 15:47:31.977716387 +0100
@@ -8,7 +8,7 @@
 DOCSHAREDIR = $(SHAREDIR)/doc/$(APPNAME)
 
 CPPFLAGS  = -std=c++11 -Wall -pipe -O3 -DNDEBUG -DwxUSE_UNICODE -DZEN_PLATFORM_OTHER -DWXINTL_NO_GETTEXT_MACRO -I. -include "zen/i18n.h" -include "zen/warn_static.h"
-LINKFLAGS =
+LINKFLAGS = -lz
 
 #distinguish Linux/OSX builds
 OPERATING_SYSTEM_NAME := $(shell uname)
