# yeah, on i386 it's still 32bit
Index: freefilesync-7.6/ui/version_check.cpp
===================================================================
--- freefilesync-7.6.orig/ui/version_check.cpp	2015-11-03 21:19:56.702256224 +0100
+++ freefilesync-7.6/ui/version_check.cpp	2015-11-03 21:19:56.698256142 +0100
@@ -421,7 +421,7 @@
 {
     if (updateCheckActive(lastUpdateCheck))
     {
-        static_assert(sizeof(time_t) >= 8, "Still using 32-bit time_t? WTF!!");
+        //static_assert(sizeof(time_t) >= 8, "Still using 32-bit time_t? WTF!!");
         const time_t now = std::time(nullptr);
         return numeric::dist(now, lastUpdateCheck) >= 7 * 24 * 3600; //check weekly
     }
