dll.h only exists in Windows
Index: freefilesync-6.12/application.cpp
===================================================================
--- freefilesync-6.12.orig/application.cpp
+++ freefilesync-6.12/application.cpp
@@ -7,7 +7,6 @@
 #include "application.h"
 #include <memory>
 #include <zen/file_access.h>
-#include <zen/dll.h>
 #include <wx/tooltip.h>
 #include <wx/log.h>
 #include <wx+/app_main.h>
Index: freefilesync-6.12/RealtimeSync/application.cpp
===================================================================
--- freefilesync-6.12.orig/RealtimeSync/application.cpp
+++ freefilesync-6.12/RealtimeSync/application.cpp
@@ -8,7 +8,9 @@
 #include "main_dlg.h"
 #include <zen/file_access.h>
 #include <zen/thread.h>
+#ifdef ZEN_WIN
 #include <zen/dll.h>
+#endif
 #include <wx/event.h>
 #include <wx/log.h>
 #include <wx/tooltip.h>
