# hardcode the install dir
Index: makehuman-1.0.0/makehuman/makehuman
===================================================================
--- makehuman-1.0.0.orig/makehuman/makehuman	2014-03-10 10:31:36.000000000 +0100
+++ makehuman-1.0.0/makehuman/makehuman	2014-03-18 20:07:46.740528296 +0100
@@ -1,7 +1,4 @@
-#!/bin/bash
-
-DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-cd $DIR
-
+#!/bin/sh
+cd /usr/share/makehuman
 python makehuman.py "$@"
 
