Index: f4l-0.2.1/src/f4lmdoc.cpp
===================================================================
--- f4l-0.2.1.orig/src/f4lmdoc.cpp	2010-12-19 13:22:53.969278003 +0100
+++ f4l-0.2.1/src/f4lmdoc.cpp	2010-12-19 13:22:57.599278003 +0100
@@ -21,7 +21,7 @@
 #include <qwidget.h>
 #include <qmessagebox.h>
 #include <qfiledialog.h>
-#include <iostream.h>
+#include <iostream>
 #include <qfiledialog.h>
 #include <qprocess.h> // reading ttf files
 #include <qimage.h>
Index: f4l-0.2.1/src/flagStonePort/transform-cxx-bsd/transform/FSVector.h
===================================================================
--- f4l-0.2.1.orig/src/flagStonePort/transform-cxx-bsd/transform/FSVector.h	2010-12-19 13:21:52.529278008 +0100
+++ f4l-0.2.1/src/flagStonePort/transform-cxx-bsd/transform/FSVector.h	2010-12-19 13:21:56.639278003 +0100
@@ -32,7 +32,7 @@
 #define __FSVECTOR_H__ 1
 
 #include "FSException.h"
-#include <new.h>
+#include <new>
 
 namespace transform
 {
Index: f4l-0.2.1/src/flagStonePort/transform-cxx-bsd/transform/FSDefineSound.h
===================================================================
--- f4l-0.2.1.orig/src/flagStonePort/transform-cxx-bsd/transform/FSDefineSound.h	2010-12-19 13:29:51.349277999 +0100
+++ f4l-0.2.1/src/flagStonePort/transform-cxx-bsd/transform/FSDefineSound.h	2010-12-19 13:30:16.439278003 +0100
@@ -137,7 +137,7 @@
             @param bytes the sound data.
             @param length the number of bytes in the sound data.
             */
-        FSDefineSound::FSDefineSound(int anIdentifier, int encoding, int rate, int channels, int sampleSize, int count, byte* bytes, size_t length);
+        FSDefineSound(int anIdentifier, int encoding, int rate, int channels, int sampleSize, int count, byte* bytes, size_t length);
 
         FSDefineSound(const FSDefineSound& rhs);
 
Index: f4l-0.2.1/src/canvasItem.h
===================================================================
--- f4l-0.2.1.orig/src/canvasItem.h	2010-12-19 13:32:26.599278002 +0100
+++ f4l-0.2.1/src/canvasItem.h	2010-12-19 13:32:39.849278003 +0100
@@ -18,7 +18,7 @@
 #ifndef CCANVASITEM_H
 #define CCANVASITEM_H
 
-#include <qcanvas.h>
+#include <qt3/qcanvas.h>
 #include <qpointarray.h>
 
 /**
