# set client ID. Generated own ID from twitch
Index: chatty-0.8.2/src/chatty/Chatty.java
===================================================================
--- chatty-0.8.2.orig/src/chatty/Chatty.java	2016-04-26 22:44:54.170419083 +0200
+++ chatty-0.8.2/src/chatty/Chatty.java	2016-04-26 22:45:14.438836575 +0200
@@ -42,7 +42,7 @@
      * If you compile this program yourself, you should create your own client
      * id on http://www.twitch.tv/kraken/oauth2/clients/new
      */
-    public static final String CLIENT_ID = "spyiu9jqdnfjtwv6l1xjk5zgt8qb91l";
+    public static final String CLIENT_ID = "q1b6bqsamkie9jl3917hsvaq388psra";
     
     /**
      * The redirect URI for getting a token.
