# set client ID. Generated own ID from twitch
Index: chatty-0.5.1/src/chatty/Chatty.java
===================================================================
--- chatty-0.5.1.orig/src/chatty/Chatty.java	2014-02-06 19:21:36.000000000 +0100
+++ chatty-0.5.1/src/chatty/Chatty.java	2014-02-06 19:23:38.021259642 +0100
@@ -20,7 +20,7 @@
     /**
      * The Twitch client id of this program.
      */
-    public static final String CLIENT_ID = <...>;
+    public static final String CLIENT_ID = "q1b6bqsamkie9jl3917hsvaq388psra";
     
     /**
      * The redirect URI for getting a token.
