# change config directory to $HOME/.config/chatty
Index: chatty-0.6.7/src/chatty/Chatty.java
===================================================================
--- chatty-0.6.7.orig/src/chatty/Chatty.java	2014-07-20 23:41:57.628118695 +0200
+++ chatty-0.6.7/src/chatty/Chatty.java	2014-07-20 23:41:57.624118695 +0200
@@ -137,7 +137,9 @@
         }
         String dir = System.getProperty("user.home")
                 + File.separator 
-                + ".chatty" 
+                + ".config" 
+                + File.separator 
+                + "chatty" 
                 + File.separator;
         new File(dir).mkdirs();
         return dir;
