# change config directory to $HOME/.config/chatty
Index: chatty-0.6/src/chatty/Chatty.java
===================================================================
--- chatty-0.6.orig/src/chatty/Chatty.java	2014-02-14 17:49:11.667728434 +0100
+++ chatty-0.6/src/chatty/Chatty.java	2014-02-14 17:49:11.663728434 +0100
@@ -135,7 +135,9 @@
         }
         String dir = System.getProperty("user.home")
                 + File.separator 
-                + ".chatty" 
+                + ".config" 
+                + File.separator 
+                + "chatty" 
                 + File.separator;
         new File(dir).mkdirs();
         return dir;
