# change config directory to $HOME/.config/chatty
Index: chatty-0.8.3/src/chatty/Chatty.java
===================================================================
--- chatty-0.8.3.orig/src/chatty/Chatty.java	2016-06-28 16:18:42.550653108 +0200
+++ chatty-0.8.3/src/chatty/Chatty.java	2016-06-28 16:18:42.546652977 +0200
@@ -188,12 +188,11 @@
      * @return
      */
     public static String getUserDataDirectory() {
-        if (useCurrentDirectory) {
-            return System.getProperty("user.dir") + File.separator;
-        }
         String dir = System.getProperty("user.home")
-                + File.separator 
-                + ".chatty" 
+                + File.separator
+                + ".config"
+                + File.separator
+                + "chatty"
                 + File.separator;
         new File(dir).mkdirs();
         return dir;
