# disable version check. updates come via apt-get
Index: chatty-0.6/src/chatty/Version.java
===================================================================
--- chatty-0.6.orig/src/chatty/Version.java	2014-02-14 17:50:18.307726451 +0100
+++ chatty-0.6/src/chatty/Version.java	2014-02-14 17:50:18.303726451 +0100
@@ -32,8 +32,8 @@
     }
     
     private void checkForNewVersion() {
-        LOGGER.info("Checking for new version..");
-        new Thread(new VersionChecker()).start();
+        //LOGGER.info("Checking for new version..");
+        //new Thread(new VersionChecker()).start();
     }
     
     /**
Index: chatty-0.6/src/chatty/TwitchClient.java
===================================================================
--- chatty-0.6.orig/src/chatty/TwitchClient.java	2014-02-14 17:50:18.307726451 +0100
+++ chatty-0.6/src/chatty/TwitchClient.java	2014-02-14 17:51:47.459723799 +0100
@@ -187,7 +187,7 @@
         
         // Check version, if enabled in this build
         if (Chatty.VERSION_CHECK_ENABLED) {
-            checkNewVersion();
+            //checkNewVersion();
         }
         
         // Connect or open connect dialog
@@ -235,7 +235,7 @@
      * Checks for a new version if the last check was long enough ago.
      */
     private void checkNewVersion() {
-        if (!settings.getBoolean("checkNewVersion")) {
+        /*if (!settings.getBoolean("checkNewVersion")) {
             return;
         }
         long ago = System.currentTimeMillis() - settings.getLong("versionLastChecked");
@@ -259,7 +259,7 @@
                     g.printSystem("You already have the newest version.");
                 }
             }
-        });
+        });*/
     }
     
 
Index: chatty-0.6/src/chatty/Chatty.java
===================================================================
--- chatty-0.6.orig/src/chatty/Chatty.java	2014-02-14 17:49:18.639728226 +0100
+++ chatty-0.6/src/chatty/Chatty.java	2014-02-14 17:52:15.143722976 +0100
@@ -51,7 +51,7 @@
      * Enable Version Checker (if you compile and distribute this yourself, you
      * may want to disable this)
      */
-    public static final boolean VERSION_CHECK_ENABLED = true;
+    public static final boolean VERSION_CHECK_ENABLED = false;
     
     /**
      * The regular URL of the textfile where the most recent version is stored.
