# global hotkeys only work on windows
Index: chatty-0.5.1/src/chatty/Chatty.java
===================================================================
--- chatty-0.5.1.orig/src/chatty/Chatty.java	2014-02-02 01:05:38.025905023 +0100
+++ chatty-0.5.1/src/chatty/Chatty.java	2014-02-02 02:24:53.801763551 +0100
@@ -13,7 +13,7 @@
 public class Chatty {
     
     public static final boolean DEBUG = false;
-    public static final boolean HOTKEY = true;
+    public static final boolean HOTKEY = false;
     
     public static final String WEBSITE =
             "http://getchatty.sourceforge.net";
