# disable version check. updates come via apt-get
Index: chatty-0.8/src/chatty/Version.java
===================================================================
--- chatty-0.8.orig/src/chatty/Version.java	2015-06-09 20:22:23.132774666 +0200
+++ chatty-0.8/src/chatty/Version.java	2015-06-09 20:22:23.128774683 +0200
@@ -32,8 +32,8 @@
     }
     
     private void checkForNewVersion() {
-        LOGGER.info("Checking for new version..");
-        new Thread(new VersionChecker()).start();
+        //LOGGER.info("Checking for new version..");
+        //new Thread(new VersionChecker()).start();
     }
     
     /**
Index: chatty-0.8/src/chatty/TwitchClient.java
===================================================================
--- chatty-0.8.orig/src/chatty/TwitchClient.java	2015-06-09 20:22:23.132774666 +0200
+++ chatty-0.8/src/chatty/TwitchClient.java	2015-06-09 20:22:23.128774683 +0200
@@ -244,7 +244,7 @@
         
         // Check version, if enabled in this build
         if (Chatty.VERSION_CHECK_ENABLED) {
-            checkNewVersion();
+            //checkNewVersion();
         }
 
         // Connect or open connect dialog
@@ -328,7 +328,7 @@
      */
     private void checkNewVersion() {
         //g.setUpdateAvailable();
-        if (!settings.getBoolean("checkNewVersion")) {
+        /*if (!settings.getBoolean("checkNewVersion")) {
             return;
         }
         long ago = System.currentTimeMillis() - settings.getLong("versionLastChecked");
@@ -354,7 +354,7 @@
                     g.printSystem("You already have the newest version.");
                 }
             }
-        });
+        });*/
     }
     
     /**
Index: chatty-0.8/src/chatty/Chatty.java
===================================================================
--- chatty-0.8.orig/src/chatty/Chatty.java	2015-06-09 20:22:23.132774666 +0200
+++ chatty-0.8/src/chatty/Chatty.java	2015-06-09 20:22:23.128774683 +0200
@@ -53,7 +53,7 @@
      * Enable Version Checker (if you compile and distribute this yourself, you
      * may want to disable this)
      */
-    public static final boolean VERSION_CHECK_ENABLED = true;
+    public static final boolean VERSION_CHECK_ENABLED = false;
     
     /**
      * The regular URL of the textfile where the most recent version is stored.
