Add remote_fs as Required Start and Stop.
Index: webcamstudio-0.56/etc/init.d/webcamstudio
===================================================================
--- webcamstudio-0.56.orig/etc/init.d/webcamstudio	2011-10-21 20:07:44.132696019 +0200
+++ webcamstudio-0.56/etc/init.d/webcamstudio	2011-10-21 20:08:08.016695309 +0200
@@ -12,8 +12,8 @@
 #
 ### BEGIN INIT INFO
 # Provides:       webcamstudio
-# Required-Start: $syslog
-# Required-Stop:
+# Required-Start: $remote_fs $syslog
+# Required-Stop:  $remote_fs
 # Default-Start:  2 3 4 5
 # Default-Stop:
 # Short-Description: WebcamStudio kernel module
