=== modified file 'virtManager/manager.py'
Index: virt-manager-1.0.1/virtManager/engine.py
===================================================================
--- virt-manager-1.0.1.orig/virtManager/engine.py	2014-02-28 19:01:44.000000000 +0100
+++ virt-manager-1.0.1/virtManager/engine.py	2014-04-18 16:58:26.988913869 +0200
@@ -635,7 +635,10 @@
                           "regular user. Try running as root.")
                 show_errmsg = False
             elif re.search(r"libvirt-sock", tb):
-                hint += _("Verify that the 'libvirtd' daemon is running.")
+                hint += _("Verify that:\n" +
+                         " - The 'libvirt-bin' package is installed\n" +
+                         " - The 'libvirtd' daemon has been started" +
+                         (conn.is_qemu_system() and "\n - You are member of the 'libvirtd' group" or ''))
                 show_errmsg = False
 
         probe_connection = self.conns[conn.get_uri()]["probeConnection"]
