=== modified file 'virtManager/manager.py'
Index: virt-manager-1.1.0/virtManager/engine.py
===================================================================
--- virt-manager-1.1.0.orig/virtManager/engine.py	2014-09-09 23:09:25.332571791 +0200
+++ virt-manager-1.1.0/virtManager/engine.py	2014-09-09 23:09:25.328571791 +0200
@@ -606,7 +606,10 @@
                           "regular user. Try running as root.")
                 show_errmsg = False
             elif re.search(r"libvirt-sock", tb):
-                hint += _("Verify that the 'libvirtd' daemon is running.")
+                hint += _("Verify that:\n" +
+                         " - The 'libvirt-bin' package is installed\n" +
+                         " - The 'libvirtd' daemon has been started" +
+                         (conn.is_qemu_system() and "\n - You are member of the 'libvirtd' group" or ''))
                 show_errmsg = False
 
         probe_connection = self.conns[conn.get_uri()]["probeConnection"]
