# fix line endings
# fix typos
Index: libqtav-1.9.0/examples/QMLPlayer/QMLPlayer_harmattan.desktop
===================================================================
--- libqtav-1.9.0.orig/examples/QMLPlayer/QMLPlayer_harmattan.desktop	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/examples/QMLPlayer/QMLPlayer_harmattan.desktop	2016-02-23 22:48:13.556226418 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
 Version=1.0
 Type=Application
 Terminal=false
 Name=QMLPlayer
-Exec=/usr/bin/single-instance /opt/QMLPlayer/bin/QMLPlayer
+Exec=/usr/bin/QMLPlayer
 Icon=/usr/share/icons/hicolor/80x80/apps/QMLPlayer80.png
 X-Window-Icon=
 X-HildonDesk-ShowInToolbar=true
Index: libqtav-1.9.0/qtc_packaging/debian_generic/QMLPlayer.desktop
===================================================================
--- libqtav-1.9.0.orig/qtc_packaging/debian_generic/QMLPlayer.desktop	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/qtc_packaging/debian_generic/QMLPlayer.desktop	2016-02-23 22:48:24.836439116 +0100
@@ -1,13 +1,13 @@
-[Desktop Entry]
-Type=Application
-Name=QtAV QML Player
-GenericName=QtAV QML player
-Comment=QtAV Multimedia Player
-Comment[zh]=多媒体播放器
-Icon=QtAV
-TryExec=/usr/bin/QMLPlayer
-Exec=QMLPlayer -f %U
-Terminal=false
-Categories=Qt;AudioVideo;Audio;Video;Player;TV;
-MimeType=application/ogg;application/x-ogg;application/sdp;application/smil;application/x-smil;application/streamingmedia;application/x-streamingmedia;application/vnd.rn-realmedia;application/vnd.rn-realmedia-vbr;audio/aac;audio/x-aac;audio/m4a;audio/x-m4a;audio/mp1;audio/x-mp1;audio/mp2;audio/x-mp2;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/mpegurl;audio/x-mpegurl;audio/mpg;audio/x-mpg;audio/rn-mpeg;audio/scpls;audio/x-scpls;audio/vnd.rn-realaudio;audio/wav;audio/x-pn-windows-pcm;audio/x-realaudio;audio/x-pn-realaudio;audio/x-ms-wma;audio/x-pls;audio/x-wav;video/mpeg;video/x-mpeg;video/x-mpeg2;video/mp4;video/msvideo;video/x-msvideo;video/quicktime;video/vnd.rn-realvideo;video/x-ms-afs;video/x-ms-asf;video/x-ms-wmv;video/x-ms-wmx;video/x-ms-wvxvideo;video/x-avi;video/x-fli;video/x-flv;video/x-theora;video/x-matroska;video/webm;audio/x-flac;audio/x-vorbis+ogg;video/x-ogm+ogg;audio/x-shorten;audio/x-ape;audio/x-wavpack;audio/x-tta;audio/AMR;audio/ac3;video/mp2t;audio/flac;
-X-KDE-Protocols=ftp,http,https,mms,rtmp,rtsp,sftp,smb
+[Desktop Entry]
+Type=Application
+Name=QtAV QML Player
+GenericName=QtAV QML player
+Comment=QtAV Multimedia Player
+Comment[zh]=多媒体播放器
+Icon=QtAV
+TryExec=/usr/bin/QMLPlayer
+Exec=QMLPlayer -f %U
+Terminal=false
+Categories=Qt;AudioVideo;Audio;Video;Player;TV;
+MimeType=application/ogg;application/x-ogg;application/sdp;application/smil;application/x-smil;application/streamingmedia;application/x-streamingmedia;application/vnd.rn-realmedia;application/vnd.rn-realmedia-vbr;audio/aac;audio/x-aac;audio/m4a;audio/x-m4a;audio/mp1;audio/x-mp1;audio/mp2;audio/x-mp2;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/mpegurl;audio/x-mpegurl;audio/mpg;audio/x-mpg;audio/rn-mpeg;audio/scpls;audio/x-scpls;audio/vnd.rn-realaudio;audio/wav;audio/x-pn-windows-pcm;audio/x-realaudio;audio/x-pn-realaudio;audio/x-ms-wma;audio/x-pls;audio/x-wav;video/mpeg;video/x-mpeg;video/x-mpeg2;video/mp4;video/msvideo;video/x-msvideo;video/quicktime;video/vnd.rn-realvideo;video/x-ms-afs;video/x-ms-asf;video/x-ms-wmv;video/x-ms-wmx;video/x-ms-wvxvideo;video/x-avi;video/x-fli;video/x-flv;video/x-theora;video/x-matroska;video/webm;audio/x-flac;audio/x-vorbis+ogg;video/x-ogm+ogg;audio/x-shorten;audio/x-ape;audio/x-wavpack;audio/x-tta;audio/AMR;audio/ac3;video/mp2t;audio/flac;
+X-KDE-Protocols=ftp,http,https,mms,rtmp,rtsp,sftp,smb
Index: libqtav-1.9.0/common.pri
===================================================================
--- libqtav-1.9.0.orig/common.pri	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/common.pri	2016-02-23 22:50:01.990282936 +0100
@@ -351,7 +351,7 @@
 # - control chars & space
 # - the windows shell meta chars "&()<>^|
 # - the potential separators ,;=
-#TODO: how to deal with  "^", "|"? every char are seperated by "|"?
+#TODO: how to deal with  "^", "|"? every char are separated by "|"?
 #how to avoid replacing "^" again for the second time
     isEmpty(1):error("shell_quote(arg) requires one argument.")
     special_chars = & \( \) < >
Index: libqtav-1.9.0/configure.pri
===================================================================
--- libqtav-1.9.0.orig/configure.pri	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/configure.pri	2016-02-23 22:50:12.294479652 +0100
@@ -42,7 +42,7 @@
     }
     #log("varstr: $$varstr")
 ##TODO: remove existing lines contain $$srcvar
-    #because write_file() will write 1 line for each value(seperated by space), so the value must be closed with "", then it's 1 value, not list
+    #because write_file() will write 1 line for each value(separated by space), so the value must be closed with "", then it's 1 value, not list
 #erase the existing var and value pair
     win32 {#:isEmpty(QMAKE_SH) { #windows sucks. can not access the cache
 
Index: libqtav-1.9.0/examples/common/common.cpp
===================================================================
--- libqtav-1.9.0.orig/examples/common/common.cpp	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/examples/common/common.cpp	2016-02-23 22:50:16.350557143 +0100
@@ -92,7 +92,7 @@
             ("height", 450, QLatin1String("height of player"))
             ("fullscreen", QLatin1String("fullscreen"))
             ("decoder", QLatin1String("FFmpeg"), QLatin1String("use a given decoder"))
-            ("decoders,-vd", QLatin1String("cuda;vaapi;vda;dxva;cedarv;ffmpeg"), QLatin1String("decoder name list in priority order seperated by ';'"))
+            ("decoders,-vd", QLatin1String("cuda;vaapi;vda;dxva;cedarv;ffmpeg"), QLatin1String("decoder name list in priority order separated by ';'"))
             ("file,f", QString(), QLatin1String("file or url to play"))
             ("language", QLatin1String("system"), QLatin1String("language on UI. can be 'system', 'none' and locale name e.g. zh_CN"))
             ("log", QString(), QLatin1String("log level. can be 'off', 'fatal', 'critical', 'warning', 'debug', 'all'"))
Index: libqtav-1.9.0/qml/libQmlAV.pro
===================================================================
--- libqtav-1.9.0.orig/qml/libQmlAV.pro	2016-02-23 22:41:48.417206331 +0100
+++ libqtav-1.9.0/qml/libQmlAV.pro	2016-02-23 22:50:07.934396388 +0100
@@ -43,7 +43,7 @@
 for(f, plugin.files) {
   plugin.commands += $$escape_expand(\\n\\t)$$TRY_COPY $$shell_path($$f) $$shell_path($$plugin.path)
 }
-#join values seperated by space. so quote is needed
+#join values separated by space. so quote is needed
 #plugin.commands = $$join(plugin.commands,$$escape_expand(\\n\\t))
 OTHER_FILES += $$qtav_qml.files
 #just append as a string to $$QMAKE_POST_LINK
Index: libqtav-1.9.0/qtc_packaging/debian_generic/player.desktop
===================================================================
--- libqtav-1.9.0.orig/qtc_packaging/debian_generic/player.desktop	2015-12-20 19:24:08.000000000 +0100
+++ libqtav-1.9.0/qtc_packaging/debian_generic/player.desktop	2016-02-23 23:03:16.901079779 +0100
@@ -1,13 +1,13 @@
-[Desktop Entry]
-Type=Application
-Name=QtAV Player
-GenericName=QtAV player
-Comment=QtAV Multimedia Player
-Comment[zh]=多媒体播放器
-Icon=QtAV
-TryExec=/usr/bin/player
-Exec=player -f %U
-Terminal=false
-Categories=Qt;AudioVideo;Audio;Video;Player;TV;
-MimeType=application/ogg;application/x-ogg;application/sdp;application/smil;application/x-smil;application/streamingmedia;application/x-streamingmedia;application/vnd.rn-realmedia;application/vnd.rn-realmedia-vbr;audio/aac;audio/x-aac;audio/m4a;audio/x-m4a;audio/mp1;audio/x-mp1;audio/mp2;audio/x-mp2;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/mpegurl;audio/x-mpegurl;audio/mpg;audio/x-mpg;audio/rn-mpeg;audio/scpls;audio/x-scpls;audio/vnd.rn-realaudio;audio/wav;audio/x-pn-windows-pcm;audio/x-realaudio;audio/x-pn-realaudio;audio/x-ms-wma;audio/x-pls;audio/x-wav;video/mpeg;video/x-mpeg;video/x-mpeg2;video/mp4;video/msvideo;video/x-msvideo;video/quicktime;video/vnd.rn-realvideo;video/x-ms-afs;video/x-ms-asf;video/x-ms-wmv;video/x-ms-wmx;video/x-ms-wvxvideo;video/x-avi;video/x-fli;video/x-flv;video/x-theora;video/x-matroska;video/webm;audio/x-flac;audio/x-vorbis+ogg;video/x-ogm+ogg;audio/x-shorten;audio/x-ape;audio/x-wavpack;audio/x-tta;audio/AMR;audio/ac3;video/mp2t;audio/flac;
-X-KDE-Protocols=ftp,http,https,mms,rtmp,rtsp,sftp,smb
+[Desktop Entry]
+Type=Application
+Name=QtAV Player
+GenericName=QtAV player
+Comment=QtAV Multimedia Player
+Comment[zh]=多媒体播放器
+Icon=QtAV
+TryExec=/usr/bin/Player
+Exec=Player -f %U
+Terminal=false
+Categories=Qt;AudioVideo;Audio;Video;Player;TV;
+MimeType=application/ogg;application/x-ogg;application/sdp;application/smil;application/x-smil;application/streamingmedia;application/x-streamingmedia;application/vnd.rn-realmedia;application/vnd.rn-realmedia-vbr;audio/aac;audio/x-aac;audio/m4a;audio/x-m4a;audio/mp1;audio/x-mp1;audio/mp2;audio/x-mp2;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/mpegurl;audio/x-mpegurl;audio/mpg;audio/x-mpg;audio/rn-mpeg;audio/scpls;audio/x-scpls;audio/vnd.rn-realaudio;audio/wav;audio/x-pn-windows-pcm;audio/x-realaudio;audio/x-pn-realaudio;audio/x-ms-wma;audio/x-pls;audio/x-wav;video/mpeg;video/x-mpeg;video/x-mpeg2;video/mp4;video/msvideo;video/x-msvideo;video/quicktime;video/vnd.rn-realvideo;video/x-ms-afs;video/x-ms-asf;video/x-ms-wmv;video/x-ms-wmx;video/x-ms-wvxvideo;video/x-avi;video/x-fli;video/x-flv;video/x-theora;video/x-matroska;video/webm;audio/x-flac;audio/x-vorbis+ogg;video/x-ogm+ogg;audio/x-shorten;audio/x-ape;audio/x-wavpack;audio/x-tta;audio/AMR;audio/ac3;video/mp2t;audio/flac;
+X-KDE-Protocols=ftp,http,https,mms,rtmp,rtsp,sftp,smb
