# set shortcut F5 to check online streams
Index: livestreamer-ui-1.0/main.py
===================================================================
--- livestreamer-ui-1.0.orig/main.py	2013-09-06 23:39:49.000000000 +0200
+++ livestreamer-ui-1.0/main.py	2014-03-06 18:38:04.649279823 +0100
@@ -2,7 +2,7 @@
 import os
 import json
 
-from PySide.QtGui import QApplication, QLabel, QPushButton, QLineEdit, QWidget, QTextEdit, QGridLayout, QTableWidget, QTableWidgetItem, QHeaderView
+from PySide.QtGui import QApplication, QLabel, QPushButton, QLineEdit, QWidget, QTextEdit, QGridLayout, QTableWidget, QTableWidgetItem, QHeaderView, QKeySequence
 from PySide.QtCore import Qt
 
 
@@ -52,6 +52,9 @@
         addSelectedLink.clicked.connect( self.add_selected_link )
         removeSelectedLink.clicked.connect( self.remove_selected_link )
 
+        #set shortcut
+        checkIfOnline.setShortcut(QKeySequence(Qt.Key_F5))
+
             # set the layouts
 
         mainLayout = QGridLayout()
