# add destdir support
Index: no-more-secrets-0.2.0/Makefile
===================================================================
--- no-more-secrets-0.2.0.orig/Makefile	2016-05-19 15:55:40.000000000 +0200
+++ no-more-secrets-0.2.0/Makefile	2016-05-19 17:50:44.829501759 +0200
@@ -1,11 +1,11 @@
 # Installation directories following GNU conventions
 prefix = /usr/local
-exec_prefix = $(prefix)
+exec_prefix = $(DESTDIR)$(prefix)
 bindir = $(exec_prefix)/bin
 sbindir = $(exec_prefix)/sbin
-datarootdir = $(prefix)/share
+datarootdir = $(DESTDIR)$(prefix)/share
 datadir = $(datarootdir)
-includedir = $(prefix)/include
+includedir = $(DESTDIR)$(prefix)/include
 mandir = $(datarootdir)/man
 
 BIN=bin
