# patch the source to find the installation directory
Index: remotebox-1.7/remotebox
===================================================================
--- remotebox-1.7.orig/remotebox	2014-02-01 00:10:17.932386860 +0100
+++ remotebox-1.7/remotebox	2014-02-01 00:10:17.924386861 +0100
@@ -7,13 +7,13 @@
 use POSIX qw(ceil);
 use MIME::Base64;
 # *** PACKAGERS MAY WISH TO PATCH THIS LOCATION ***
-use lib "$Bin/share/remotebox";
+use lib "/usr/share/remotebox";
 # *************************************************
 use vboxService qw($endpoint $fault :all);
 
 # *** PACKAGERS MAY WISH TO PATCH THESE LOCATIONS ***
-our $sharedir = "$Bin/share/remotebox";
-our $docdir   = "$Bin/docs";
+our $sharedir = "/usr/share/remotebox";
+our $docdir   = "/usr/share/doc/remotebox";
 # ***************************************************
 
 require 'vboxserializers.pl';
