# patch the source to find the installation directory
Index: remotebox-1.6/remotebox
===================================================================
--- remotebox-1.6.orig/remotebox	2013-11-02 15:14:26.791393777 +0100
+++ remotebox-1.6/remotebox	2013-11-02 15:17:59.703391959 +0100
@@ -9,13 +9,13 @@
 use File::Spec;
 use MIME::Base64;
 # *** PACKAGERS MAY WISH TO PATCH THIS LOCATION ***
-use lib "$Bin/share/remotebox";
+use lib "/usr/share/remotebox";
 # *************************************************
 use vboxService qw($endpoint $fault :all);
 
 # *** PACKAGERS MAY WISH TO PATCH THESE LOCATIONS ***
-our $sharedir = "$Bin/share/remotebox";
-our $docdir   = "$Bin/docs";
+our $sharedir = "/usr/share/remotebox";
+our $docdir   = "/usr/share/doc/remotebox";
 # ***************************************************
 
 require 'vboxserializers.pl';
