# don't take files from inside the debian directory
Index: miam-player-0.8.0/src/Player/Player.pro
===================================================================
--- miam-player-0.8.0.orig/src/Player/Player.pro	2016-02-06 16:17:41.000000000 +0100
+++ miam-player-0.8.0/src/Player/Player.pro	2016-02-23 23:11:24.546177724 +0100
@@ -130,16 +130,7 @@
 }
 unix:!macx {
     target.path = /usr/bin
-    desktop.path = /usr/share/applications
-    desktop.files = $$PWD/../../debian/usr/share/applications/miam-player.desktop
-    icon64.path = /usr/share/icons/hicolor/64x64/apps
-    icon64.files = $$PWD/../../debian/usr/share/icons/hicolor/64x64/apps/application-x-miamplayer.png
-    appdata.path = /usr/share/appdata
-    appdata.files = $$PWD/../../fedora/miam-player.appdata.xml
-    INSTALLS += desktop \
-	target \
-	icon64 \
-	appdata
+    INSTALLS += target
 }
 macx {
     ICON = $$PWD/../../osx/MiamPlayer.icns
