Encoding is deprecated.
Index: wxhexeditor-0.11/resources/wxHexEditor.desktop
===================================================================
--- wxhexeditor-0.11.orig/resources/wxHexEditor.desktop	2011-07-01 22:48:12.223565861 +0200
+++ wxhexeditor-0.11/resources/wxHexEditor.desktop	2011-07-01 22:48:18.043565861 +0200
@@ -1,6 +1,5 @@
 [Desktop Entry]
 Version=v1.0
-Encoding=UTF-8
 Name=wxHexEditor
 GenericName=wxHexEditor
 Comment=A hex editor for huge files and devices
