Description: add x-ubuntu-gettext-domain to .desktop file
Bug-Ubuntu: https://bugs.launchpad.net/bugs/514272
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Last-Update: 2014-02-13

--- a/liferea.desktop.in
+++ b/liferea.desktop.in
@@ -10,5 +10,6 @@
 Terminal=false
 Type=Application
 Categories=Network;Feed;
+X-Ubuntu-Gettext-Domain=liferea
 Version=1.1
 MimeType=application/rss+xml;application/atom+xml;application/rdf+xml;
