Update plugin.
Index: xvst-2.4.1/resources/services/pornhub/pornhub.js
===================================================================
--- xvst-2.4.1.orig/resources/services/pornhub/pornhub.js	2009-11-20 20:53:30.000000000 +0100
+++ xvst-2.4.1/resources/services/pornhub/pornhub.js	2012-08-03 19:09:59.513235979 +0200
@@ -25,9 +25,9 @@
 
 function RegistVideoService()
 {
-	this.version = "1.0.1";
+	this.version = "1.0.3";
 	this.minVersion = "2.0.0a";
-	this.author = "Xesc & Technology 2009";
+	this.author = "Xesc & Technology 2012";
 	this.website = "http://www.pornhub.com/";
 	this.ID = "pornhub.com";
 	this.caption = "PornHub";
@@ -37,19 +37,15 @@
 
 function getVideoInformation(url)
 {
-	// video information
-	var result = new VideoDefinition();
-	// download webpage
-	var http = new Http();
-	var html = http.downloadWebpage(url);
-	// get video title
-	result.title = copyBetween(html, "<title>", "- Pornhub.com</title>");
-	// get xml url
-	var xmlUrl = cleanUrl(copyBetween(html, 'options=', '"'));
-	// download xml
-	var xml = http.downloadWebpage(xmlUrl);
-	// get video url
-	result.URL = copyBetween(xml, "<flv_url>", "</flv_url>");
-	// return the video information
-	return result;
-}
+    // video information
+    var result = new VideoDefinition();
+    // download webpage
+    var http = new Http();
+    var html = http.downloadWebpage(url);
+    // get video title
+    result.title = copyBetween(html, "<title>", "- Pornhub.com</title>");
+    // get video url
+    result.URL = cleanUrl(copyBetween(html, '"video_url":"', '","'));
+    // return the video information
+    return result;
+}
\ No newline at end of file
