Update plugin
Index: xvst-2.4.1/resources/services/xhamster/xhamster.js
===================================================================
--- xvst-2.4.1.orig/resources/services/xhamster/xhamster.js	2012-02-14 18:52:49.127910850 +0100
+++ xvst-2.4.1/resources/services/xhamster/xhamster.js	2012-02-14 18:53:01.231910489 +0100
@@ -25,7 +25,7 @@
 
 function RegistVideoService()
 {
-	this.version = "1.0.2";
+	this.version = "1.0.3";
 	this.minVersion = "2.0.0a";
 	this.author = "Supelex Technologies (mantained by Xesc & Technology 2010)";
 	this.website = "http://www.xhamster.com/";
@@ -37,7 +37,7 @@
 
 function getVideoInformation(url)
 {
-	const FLV_URL = "%1/flv2/%2";	
+	const FLV_URL = "%1/key=%2";	
 	// video information
 	var result = new VideoDefinition();
 	// download webpage
@@ -46,9 +46,10 @@
 	// get video title
 	result.title = copyBetween(html, "<title>", "</title>");
 	// get the flv url
-	var srv = copyBetween(html, '\'srv\': \'', '\',');
-	var file = copyBetween(html, '\'file\': \'', '\',');
-	result.URL = strFormat(FLV_URL, srv, file);
+	var srv = /'srv'\s*:\s*'(.*?)'/
+	var file = /'file'\s*:\s*'(.*?)'/
+	// build final URL
+	result.URL = strFormat(FLV_URL, srv.exec(html)[1], file.exec(html)[1]);
 	// return the video information
 	return result;
 }
