Update youporn plugin.
Index: xvst-2.4.1/resources/services/youporn/youporn.js
===================================================================
--- xvst-2.4.1.orig/resources/services/youporn/youporn.js	2012-02-14 18:50:01.811915824 +0100
+++ xvst-2.4.1/resources/services/youporn/youporn.js	2012-02-14 18:50:56.627914196 +0100
@@ -3,7 +3,7 @@
 * This file is part of xVideoServiceThief,
 * an open-source cross-platform Video service download
 *
-* Copyright (C) 2007 - 2009 Xesc & Technology
+* Copyright (C) 2007 - 2012 Xesc & Technology
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
@@ -25,9 +25,9 @@
 
 function RegistVideoService()
 {
-	this.version = "2.0.3";
+	this.version = "2.0.5";
 	this.minVersion = "2.0.0a";
-	this.author = "Xesc & Technology 2009 (thanks to Matze Ba)";
+	this.author = "Xesc & Technology 2012 (thanks to Matze Ba)";
 	this.website = "http://www.youporn.com/";
 	this.ID = "youporn.com";
 	this.caption = "YouPorn";
@@ -47,18 +47,21 @@
 	// get video title
 	result.title = copyBetween(html, "<title>", "- Free Porn Videos");
 	// get the flv url
-	result.URL = copyBetween(html, "<div id=\"download\">", "</div");
-	result.URL = copyBetween(result.URL, "<a href=\"", "\"");
+	var video_url = /'video_url'\s*:\s*'(.*?)'/
+	result.URL = video_url.exec(html)[1];
+	result.URL = strReplace(result.URL, "%26", "&");
+	// get cookies
+	result.cookies = http.getCookies("|");
 	// return the video information
 	return result;
 }
 
 function searchVideos(keyWord, pageIndex)
 {
-	const URL_SEARCH = "http://youporn.com/search?query=%1&type=straight&page=%2";
-	const HTML_SEARCH_START = '<div id="video-listing">';
-	const HTML_SEARCH_FINISH = '<div id="pages" class="clearfix">';
-	const HTML_SEARCH_SEPARATOR = "5.00</span>";
+	const URL_SEARCH = "http://www.youporn.com/search/?query=%1&type=straight&page=%2";
+	const HTML_SEARCH_START = '<div class="videoList"';
+	const HTML_SEARCH_FINISH = 'id="related-searches"';
+	const HTML_SEARCH_SEPARATOR = '</li>';
 	// replace all spaces for "+"
 	keyWord = strReplace(keyWord, " ", "+");
 	// init search results object
@@ -66,16 +69,14 @@
 	// init http object
 	var http = new Http();
 	var html = http.downloadWebpage(strFormat(URL_SEARCH, keyWord, pageIndex) + "&user_choice=Enter");
-	// get the search summary
-	var summary = copyBetween(html, '<div id="search-listing-hd">', '<div id="search-sort" class="clearfix">'); 
-	summary = copyBetween(summary, '<h1>', '</h1>');
-	searchResults.setSummary(cleanSummary(summary));
+	// no summary avaiable
+	searchResults.setSummary("");
 	// get results html block
 	var htmlResults = copyBetween(html, HTML_SEARCH_START, HTML_SEARCH_FINISH);
 	// if we found some results then...
 	if (htmlResults != "")
 	{
-		var blocks = splitString(htmlResults,HTML_SEARCH_SEPARATOR);
+		var blocks = splitString(htmlResults, HTML_SEARCH_SEPARATOR);
 		for (n = 0; n < blocks.length-1; n++)
 			parseResultItem(searchResults, blocks[n]);
 	}
@@ -83,19 +84,6 @@
 	return searchResults;
 }
 
-function cleanSummary(summary)
-{
-	// remove all "\n"
-	summary = strReplace(summary, "\n", "");
-	// remove unused tags
-	summary = strReplace(summary, "</span>", '');
-	summary = strReplace(summary, "<span>", '');
-	summary = strReplace(summary, "<em>", '');
-	summary = strReplace(summary, "</em>", '');
-	// return cleanned summary
-	return summary;
-}
-
 function parseResultItem(searchResults, html)
 {
 	const VIDEO_URL = "http://youporn.com";
@@ -104,18 +92,18 @@
 	// get video url
 	videoUrl = VIDEO_URL + copyBetween(html, 'href="', '"');
 	// get title and image url
-	tmp = copyBetween(html, '<img id="', '/>');
+	tmp = copyBetween(html, '<img', '>');
 	title = copyBetween(tmp, 'alt="', '"');
 	imageUrl = copyBetween(tmp, 'src="', '"');
 	// get video description
-	description = "";
+	description = copyBetween(html, '<div class="views">', '</div>');
 	// get video duration
-	tmp = copyBetween(html, '<div class="duration_views">', '</div>');
-	tmp = copyBetween(tmp, '<h2>', '</h2>');
+	tmp = copyBetween(html, '<h2 class="duration">', '</h2>');
 	duration = convertToSeconds(tmp); 
 	// get rating
-	tmp = copyBetween(html, '<div class="rating">', 'span>');
-	rating = copyBetween(tmp, '<p>', '<');
+	tmp = copyBetween(html, '<div class="rating">', '</div>');
+	tmp = copyBetween(tmp, 'class="stars star-', '"');
+	rating = strReplace(tmp, "-", ".");
 	// add to results list
 	searchResults.addSearchResult(videoUrl, imageUrl, title, description, duration, rating);
 }
