Fix Category name.
Index: pac-4.5.3.7.2/res/pac.desktop
===================================================================
--- pac-4.5.3.7.2.orig/res/pac.desktop	2014-04-05 21:29:12.222683928 +0200
+++ pac-4.5.3.7.2/res/pac.desktop	2014-04-05 21:29:12.218683928 +0200
@@ -5,10 +5,9 @@
 Icon=pac
 Type=Application
 Exec=/usr/bin/pac
-StartupNotify=true
 Name[en_US]=PAC
 Comment[en_US]=Perl Auto Connector
-Categories=Applications;Network;
+Categories=Application;Network;
 X-GNOME-Autostart-enabled=false
 Actions=Shell;Quick;
 
