Fix Category name.
Index: pac-2.5.5.6/res/pac.desktop
===================================================================
--- pac-2.5.5.6.orig/res/pac.desktop	2011-03-06 10:55:30.366270998 +0100
+++ pac-2.5.5.6/res/pac.desktop	2011-03-06 10:55:53.306271001 +0100
@@ -5,9 +5,8 @@
 Icon=pac
 Type=Application
 Exec=/usr/bin/pac
-StartupNotify=true
 Name[en_US]=PAC
 Comment[en_US]=Perl Auto Connector
-Categories=Applications;Network;
+Categories=Application;Network;
 X-GNOME-Autostart-enabled=false
 
