There should be code under an if.
--- a/src/etc/templates/startup.sh.template
+++ b/src/etc/templates/startup.sh.template
@@ -47,8 +47,9 @@
 # in the base directory as an indication. After the daemon has run
 # the startup script any such file will be deleted
 #-------------------------------------------------------------------
-if test "$autoshutdown" = "yes"; then
+if test "$autoshutdown" = "yes" ; then
     # Actions only done when the daemon was automatically shutdown
+    echo "autoshutdown=yes"
 fi
 
 # Normal exit from the script. In case of error use "exit 1"
