Index: lince-1.2/lib/Makefile.in
===================================================================
--- lince-1.2.orig/lib/Makefile.in	2010-01-10 13:58:41.885661473 +0100
+++ lince-1.2/lib/Makefile.in	2010-01-10 14:26:14.198161132 +0100
@@ -88,10 +88,10 @@
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
 am__mv = mv -f
-CXXCOMPILE = $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) \
+CXXCOMPILE = $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) \
 	$(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS)
 LTCXXCOMPILE = $(LIBTOOL) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) \
-	--mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) \
+	--mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) \
 	$(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS)
 CXXLD = $(CXX)
 CXXLINK = $(LIBTOOL) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) \
@@ -171,7 +171,6 @@
 DATADIRNAME = @DATADIRNAME@
 DBUS_CFLAGS = @DBUS_CFLAGS@
 DBUS_LIBS = @DBUS_LIBS@
-DEFS = @DEFS@
 DEPDIR = @DEPDIR@
 DISTCHECK_CONFIGURE_FLAGS = @DISTCHECK_CONFIGURE_FLAGS@
 DSYMUTIL = @DSYMUTIL@
@@ -433,74 +432,74 @@
 @am__fastdepCXX_FALSE@	$(LTCXXCOMPILE) -c -o $@ $<
 
 liblince_1_la-AlertsManager.lo: AlertsManager.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-AlertsManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-AlertsManager.Tpo -c -o liblince_1_la-AlertsManager.lo `test -f 'AlertsManager.cc' || echo '$(srcdir)/'`AlertsManager.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-AlertsManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-AlertsManager.Tpo -c -o liblince_1_la-AlertsManager.lo `test -f 'AlertsManager.cc' || echo '$(srcdir)/'`AlertsManager.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-AlertsManager.Tpo $(DEPDIR)/liblince_1_la-AlertsManager.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='AlertsManager.cc' object='liblince_1_la-AlertsManager.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-AlertsManager.lo `test -f 'AlertsManager.cc' || echo '$(srcdir)/'`AlertsManager.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-AlertsManager.lo `test -f 'AlertsManager.cc' || echo '$(srcdir)/'`AlertsManager.cc
 
 liblince_1_la-Core.lo: Core.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Core.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Core.Tpo -c -o liblince_1_la-Core.lo `test -f 'Core.cc' || echo '$(srcdir)/'`Core.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Core.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Core.Tpo -c -o liblince_1_la-Core.lo `test -f 'Core.cc' || echo '$(srcdir)/'`Core.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-Core.Tpo $(DEPDIR)/liblince_1_la-Core.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='Core.cc' object='liblince_1_la-Core.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Core.lo `test -f 'Core.cc' || echo '$(srcdir)/'`Core.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Core.lo `test -f 'Core.cc' || echo '$(srcdir)/'`Core.cc
 
 liblince_1_la-IpFilterManager.lo: IpFilterManager.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-IpFilterManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-IpFilterManager.Tpo -c -o liblince_1_la-IpFilterManager.lo `test -f 'IpFilterManager.cc' || echo '$(srcdir)/'`IpFilterManager.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-IpFilterManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-IpFilterManager.Tpo -c -o liblince_1_la-IpFilterManager.lo `test -f 'IpFilterManager.cc' || echo '$(srcdir)/'`IpFilterManager.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-IpFilterManager.Tpo $(DEPDIR)/liblince_1_la-IpFilterManager.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='IpFilterManager.cc' object='liblince_1_la-IpFilterManager.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-IpFilterManager.lo `test -f 'IpFilterManager.cc' || echo '$(srcdir)/'`IpFilterManager.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-IpFilterManager.lo `test -f 'IpFilterManager.cc' || echo '$(srcdir)/'`IpFilterManager.cc
 
 liblince_1_la-SessionManager.lo: SessionManager.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-SessionManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-SessionManager.Tpo -c -o liblince_1_la-SessionManager.lo `test -f 'SessionManager.cc' || echo '$(srcdir)/'`SessionManager.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-SessionManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-SessionManager.Tpo -c -o liblince_1_la-SessionManager.lo `test -f 'SessionManager.cc' || echo '$(srcdir)/'`SessionManager.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-SessionManager.Tpo $(DEPDIR)/liblince_1_la-SessionManager.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='SessionManager.cc' object='liblince_1_la-SessionManager.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-SessionManager.lo `test -f 'SessionManager.cc' || echo '$(srcdir)/'`SessionManager.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-SessionManager.lo `test -f 'SessionManager.cc' || echo '$(srcdir)/'`SessionManager.cc
 
 liblince_1_la-SetupManager.lo: SetupManager.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-SetupManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-SetupManager.Tpo -c -o liblince_1_la-SetupManager.lo `test -f 'SetupManager.cc' || echo '$(srcdir)/'`SetupManager.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-SetupManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-SetupManager.Tpo -c -o liblince_1_la-SetupManager.lo `test -f 'SetupManager.cc' || echo '$(srcdir)/'`SetupManager.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-SetupManager.Tpo $(DEPDIR)/liblince_1_la-SetupManager.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='SetupManager.cc' object='liblince_1_la-SetupManager.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-SetupManager.lo `test -f 'SetupManager.cc' || echo '$(srcdir)/'`SetupManager.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-SetupManager.lo `test -f 'SetupManager.cc' || echo '$(srcdir)/'`SetupManager.cc
 
 liblince_1_la-Torrent.lo: Torrent.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Torrent.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Torrent.Tpo -c -o liblince_1_la-Torrent.lo `test -f 'Torrent.cc' || echo '$(srcdir)/'`Torrent.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Torrent.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Torrent.Tpo -c -o liblince_1_la-Torrent.lo `test -f 'Torrent.cc' || echo '$(srcdir)/'`Torrent.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-Torrent.Tpo $(DEPDIR)/liblince_1_la-Torrent.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='Torrent.cc' object='liblince_1_la-Torrent.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Torrent.lo `test -f 'Torrent.cc' || echo '$(srcdir)/'`Torrent.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Torrent.lo `test -f 'Torrent.cc' || echo '$(srcdir)/'`Torrent.cc
 
 liblince_1_la-TorrentsManager.lo: TorrentsManager.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-TorrentsManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-TorrentsManager.Tpo -c -o liblince_1_la-TorrentsManager.lo `test -f 'TorrentsManager.cc' || echo '$(srcdir)/'`TorrentsManager.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-TorrentsManager.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-TorrentsManager.Tpo -c -o liblince_1_la-TorrentsManager.lo `test -f 'TorrentsManager.cc' || echo '$(srcdir)/'`TorrentsManager.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-TorrentsManager.Tpo $(DEPDIR)/liblince_1_la-TorrentsManager.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='TorrentsManager.cc' object='liblince_1_la-TorrentsManager.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-TorrentsManager.lo `test -f 'TorrentsManager.cc' || echo '$(srcdir)/'`TorrentsManager.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-TorrentsManager.lo `test -f 'TorrentsManager.cc' || echo '$(srcdir)/'`TorrentsManager.cc
 
 liblince_1_la-Utils.lo: Utils.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Utils.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Utils.Tpo -c -o liblince_1_la-Utils.lo `test -f 'Utils.cc' || echo '$(srcdir)/'`Utils.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-Utils.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-Utils.Tpo -c -o liblince_1_la-Utils.lo `test -f 'Utils.cc' || echo '$(srcdir)/'`Utils.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-Utils.Tpo $(DEPDIR)/liblince_1_la-Utils.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='Utils.cc' object='liblince_1_la-Utils.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Utils.lo `test -f 'Utils.cc' || echo '$(srcdir)/'`Utils.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-Utils.lo `test -f 'Utils.cc' || echo '$(srcdir)/'`Utils.cc
 
 liblince_1_la-XMLReader.lo: XMLReader.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-XMLReader.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-XMLReader.Tpo -c -o liblince_1_la-XMLReader.lo `test -f 'XMLReader.cc' || echo '$(srcdir)/'`XMLReader.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-XMLReader.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-XMLReader.Tpo -c -o liblince_1_la-XMLReader.lo `test -f 'XMLReader.cc' || echo '$(srcdir)/'`XMLReader.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-XMLReader.Tpo $(DEPDIR)/liblince_1_la-XMLReader.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='XMLReader.cc' object='liblince_1_la-XMLReader.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-XMLReader.lo `test -f 'XMLReader.cc' || echo '$(srcdir)/'`XMLReader.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-XMLReader.lo `test -f 'XMLReader.cc' || echo '$(srcdir)/'`XMLReader.cc
 
 liblince_1_la-XMLWriter.lo: XMLWriter.cc
-@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-XMLWriter.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-XMLWriter.Tpo -c -o liblince_1_la-XMLWriter.lo `test -f 'XMLWriter.cc' || echo '$(srcdir)/'`XMLWriter.cc
+@am__fastdepCXX_TRUE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -MT liblince_1_la-XMLWriter.lo -MD -MP -MF $(DEPDIR)/liblince_1_la-XMLWriter.Tpo -c -o liblince_1_la-XMLWriter.lo `test -f 'XMLWriter.cc' || echo '$(srcdir)/'`XMLWriter.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/liblince_1_la-XMLWriter.Tpo $(DEPDIR)/liblince_1_la-XMLWriter.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='XMLWriter.cc' object='liblince_1_la-XMLWriter.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-XMLWriter.lo `test -f 'XMLWriter.cc' || echo '$(srcdir)/'`XMLWriter.cc
+@am__fastdepCXX_FALSE@	$(LIBTOOL)  --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX)  $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(liblince_1_la_CXXFLAGS) $(CXXFLAGS) -c -o liblince_1_la-XMLWriter.lo `test -f 'XMLWriter.cc' || echo '$(srcdir)/'`XMLWriter.cc
 
 mostlyclean-libtool:
 	-rm -f *.lo
Index: lince-1.2/lib/Lince/Makefile.in
===================================================================
--- lince-1.2.orig/lib/Lince/Makefile.in	2010-01-10 14:26:21.405682879 +0100
+++ lince-1.2/lib/Lince/Makefile.in	2010-01-10 14:26:38.605668021 +0100
@@ -107,7 +107,6 @@
 DATADIRNAME = @DATADIRNAME@
 DBUS_CFLAGS = @DBUS_CFLAGS@
 DBUS_LIBS = @DBUS_LIBS@
-DEFS = @DEFS@
 DEPDIR = @DEPDIR@
 DISTCHECK_CONFIGURE_FLAGS = @DISTCHECK_CONFIGURE_FLAGS@
 DSYMUTIL = @DSYMUTIL@
