# remove parameter
# the candidate does not have one, whatever.
Index: lince-1.3/src/LoadTorrentDialog.cc
===================================================================
--- lince-1.3.orig/src/LoadTorrentDialog.cc	2014-03-29 16:38:53.229690081 +0100
+++ lince-1.3/src/LoadTorrentDialog.cc	2014-03-29 16:52:43.189682996 +0100
@@ -498,7 +498,7 @@
 		Utils::warningDialog(_("Selected file is not a torrent"), url);
 	}
 
-	close(fd);
+	close();
 }
 
 void LoadTorrentDialog::selectAndSetMagnet(const Glib::ustring &uri)
