Apply changes from SVN.
diff -Nur lince-1.3/configure.ac lince/configure.ac
--- lince-1.3/configure.ac	2012-06-28 13:57:28.000000000 +0200
+++ lince/configure.ac	2012-11-02 18:20:21.149299477 +0100
@@ -24,8 +24,8 @@
 IT_PROG_INTLTOOL([0.35.0])
 AC_PROG_LIBTOOL
 
-dnl Check for boost 1.36
-BOOST_REQUIRE(1.36)
+dnl Check for boost 1.49
+BOOST_REQUIRE(1.49)
 
 dnl Check for boost system
 BOOST_SYSTEM()
@@ -39,6 +39,9 @@
 dnl Check for boost date time
 BOOST_DATE_TIME()
 
+dnl Check for boost threads
+BOOST_THREADS()
+
 dnl Check for Gtkmm-3 Library
 PKG_CHECK_MODULES(GTKMM3, [gtkmm-3.0 >= 3.2.0])
 AC_SUBST(GTKMM3_CFLAGS)
diff -Nur lince-1.3/data/lince.desktop lince/data/lince.desktop
--- lince-1.3/data/lince.desktop	2012-11-02 18:07:56.665271148 +0100
+++ lince/data/lince.desktop	2012-11-02 18:06:57.220493000 +0100
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Lince
 Comment=Bittorrent client
 Comment[es]=Cliente de bittorrent
diff -Nur lince-1.3/lib/AlertsManager.cc lince/lib/AlertsManager.cc
--- lince-1.3/lib/AlertsManager.cc	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/AlertsManager.cc	2012-11-02 18:06:57.220493000 +0100
@@ -150,7 +150,7 @@
 			}
 			else if (portmap_error_alert* p = dynamic_cast<portmap_error_alert*>(a.get()))
 			{
-				if (p->type == 0)
+				if (p->type() == 0)
 					m_signal_portmap_failed(_("Failed to map NAT-PMP ports"), type);
 				else
 					m_signal_portmap_failed(_("Failed to map UPnP ports"), type);
@@ -354,7 +354,7 @@
 			{
 			}
 		}
-		catch (libtorrent::invalid_handle)
+		catch (libtorrent::invalid_handle &)
 		{
 		}
 
diff -Nur lince-1.3/lib/Lince/Torrent.h lince/lib/Lince/Torrent.h
--- lince-1.3/lib/Lince/Torrent.h	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/Lince/Torrent.h	2012-11-02 18:06:57.220493000 +0100
@@ -224,7 +224,7 @@
 			return status().next_announce;
 		};
 
-		boost::filesystem::path save_path() const { return th.save_path(); };
+		std::string save_path() const { return th.save_path(); };
 		void move_storage(std::string const& save_path) { th.move_storage(save_path); };
 
 		std::vector<libtorrent::announce_entry> const trackers() const { return th.trackers(); };
diff -Nur lince-1.3/lib/Makefile.am lince/lib/Makefile.am
--- lince-1.3/lib/Makefile.am	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/Makefile.am	2012-11-02 18:06:57.220493000 +0100
@@ -27,4 +27,5 @@
 	$(LIBTORRENT_LIBS) \
 	$(LIBXML2_LIBS) \
 	$(BOOST_DATE_TIME_LIBS) \
-	$(BOOST_SIGNALS_LIBS)
+	$(BOOST_SIGNALS_LIBS) \
+	$(BOOST_THREAD_LIBS)
diff -Nur lince-1.3/lib/SessionManager.cc lince/lib/SessionManager.cc
--- lince-1.3/lib/SessionManager.cc	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/SessionManager.cc	2012-11-02 18:06:57.220493000 +0100
@@ -556,7 +556,7 @@
 
 			writer.startElement("Path");
 			writer.addAttribute("type", "string");
-			writer.addElementData(i->save_path().native_directory_string());
+			writer.addElementData(i->save_path());
 			writer.endElement();
 
 			writer.startElement("Status");
@@ -690,7 +690,7 @@
 	boost::intrusive_ptr<torrent_info> t;
 	try
 	{
-		t = new torrent_info(torrent_path);
+		t = new torrent_info(torrent_path.native());
 	}
 	catch (std::exception&)
 	{
@@ -710,7 +710,7 @@
 
 	std::vector<char> buf;
 	error_code ec;
-	if (load_file(build_filename(get_resume_data_base_directory(), hash_str.str())\
+	if (load_file(build_filename(get_resume_data_base_directory(), hash_str.str()).native()\
 			, buf, ec) == 0)
 		p.resume_data = &buf;
 
@@ -804,7 +804,7 @@
 
 	// Store the torrent for future sessions
 	std::string filename = Lince::build_filename_as_string(
-			Lince::get_torrents_base_directory().native_file_string(), th.name());
+			Lince::get_torrents_base_directory().native(), th.name());
 	boost::filesystem::basic_ofstream<char> out;
 	Lince::save_entry(filename, torrent.create_torrent());
 
@@ -825,7 +825,7 @@
 	boost::intrusive_ptr<torrent_info> t;
 	try
 	{
-		t = new torrent_info(torrent_path);
+		t = new torrent_info(torrent_path.native());
 	}
 	catch (std::exception&)
 	{
@@ -857,8 +857,8 @@
 	th = add_torrent(p);
 
 	// Copy torrent
-	if (!exists(boost::filesystem::path(build_filename(get_torrents_base_directory(), torrent_path.leaf())))){
-		boost::filesystem::copy_file(torrent_path, boost::filesystem::path(build_filename(get_torrents_base_directory(), torrent_path.leaf())));
+	if (!exists(boost::filesystem::path(build_filename(get_torrents_base_directory(), torrent_path.filename().native())))){
+		boost::filesystem::copy_file(torrent_path, boost::filesystem::path(build_filename(get_torrents_base_directory(), torrent_path.filename().native())));
 	}
 
 	if (!priorityfiles.empty())
@@ -880,7 +880,7 @@
 
 	torrent.set_share_ratio(share_ratio);
 	torrent.set_storage_mode(p.storage_mode);
-	torrent.set_file_name(torrent_path.leaf());
+	torrent.set_file_name(torrent_path.filename().native());
 	torrent.set_file_priorities(priorityfiles);
 	torrent.set_file_new_names(torrentfilenewnames);
 
diff -Nur lince-1.3/lib/Torrent.cc lince/lib/Torrent.cc
--- lince-1.3/lib/Torrent.cc	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/Torrent.cc	2012-11-02 18:06:57.220493000 +0100
@@ -116,7 +116,7 @@
 {
 	int i;
 	libtorrent::file_entry file_entry;
-	std::vector<libtorrent::file_entry>::const_iterator file_iterator;
+	libtorrent::torrent_info::file_iterator file_iterator;
 
 	// Check that we are in a valid torrent and file priorities are set.
 	if (!th.is_valid() || file_priorities().empty()) return;
@@ -245,7 +245,9 @@
 std::string Torrent::get_date_as_string(void)
 {
 	if (get_torrent_info().creation_date())
-		return boost::posix_time::to_simple_string(get_torrent_info().creation_date().get());
+		return boost::posix_time::to_simple_string(
+				boost::posix_time::from_time_t(
+						get_torrent_info().creation_date().get()));
 
 	return "";
 };
diff -Nur lince-1.3/lib/Utils.cc lince/lib/Utils.cc
--- lince-1.3/lib/Utils.cc	2012-06-28 13:50:15.000000000 +0200
+++ lince/lib/Utils.cc	2012-11-02 18:06:57.220493000 +0100
@@ -41,7 +41,7 @@
 
 std::string Lince::build_filename_as_string(const fs::path &p, const std::string &s)
 {
-	return fs::path(p/s).native_directory_string();
+	return fs::path(p/s).native();
 }
 
 fs::path Lince::get_base_directory()
diff -Nur lince-1.3/po/ca.po lince/po/ca.po
--- lince-1.3/po/ca.po	2012-11-02 18:07:56.589271145 +0100
+++ lince/po/ca.po	2012-11-02 18:06:57.220493000 +0100
@@ -1127,11 +1127,11 @@
 msgstr "a"
 
 #: /home/toriet/po/lince/lincetorrent/src/OptionsWindows.cc:59
-msgid "Add suppport for DHT. (*)"
+msgid "Add support for DHT. (*)"
 msgstr "Afegir suport per a DHT.(*)"
 
 #: /home/toriet/po/lince/lincetorrent/src/OptionsWindows.cc:60
-msgid "Add suppport for UPNP. (*)"
+msgid "Add support for UPNP. (*)"
 msgstr "Afegir suport per a UPNP.(*)"
 
 #: /home/toriet/po/lince/lincetorrent/src/OptionsWindows.cc:61
@@ -1139,7 +1139,7 @@
 msgstr "Afegir suport per a buscar nodes en la xarxa local."
 
 #: /home/toriet/po/lince/lincetorrent/src/OptionsWindows.cc:62
-msgid "Adds suppport for exchanging metadata from peers. (*)"
+msgid "Adds support for exchanging metadata from peers. (*)"
 msgstr "Afegir suport per a intercanviar metadades entre els clients."
 
 #: /home/toriet/po/lince/lincetorrent/src/OptionsWindows.cc:63
diff -Nur lince-1.3/po/de.po lince/po/de.po
--- lince-1.3/po/de.po	2012-11-02 18:07:56.589271145 +0100
+++ lince/po/de.po	2012-11-02 18:06:57.220493000 +0100
@@ -798,11 +798,11 @@
 msgstr "bis"
 
 #: src/OptionsWindows.cc:59
-msgid "Add suppport for DHT."
+msgid "Add support for DHT."
 msgstr "DHT aktivieren"
 
 #: src/OptionsWindows.cc:60
-msgid "Add suppport for UPNP."
+msgid "Add support for UPNP."
 msgstr "UPNP aktivieren"
 
 #: src/OptionsWindows.cc:61
@@ -810,7 +810,7 @@
 msgstr "Suche Peers im lokalen Netzwerk"
 
 #: src/OptionsWindows.cc:62
-msgid "Adds suppport for exchanging metadata from peers. (*)"
+msgid "Adds support for exchanging metadata from peers. (*)"
 msgstr "Metadata mit Peers austauschen (*)"
 
 #: src/OptionsWindows.cc:63
diff -Nur lince-1.3/po/es.po lince/po/es.po
--- lince-1.3/po/es.po	2012-11-02 18:07:56.589271145 +0100
+++ lince/po/es.po	2012-11-02 18:06:57.220493000 +0100
@@ -798,11 +798,11 @@
 msgstr "a"
 
 #: src/OptionsWindows.cc:57
-msgid "Add suppport for DHT."
+msgid "Add support for DHT."
 msgstr "Añadir soporte para DHT. "
 
 #: src/OptionsWindows.cc:58
-msgid "Add suppport for UPNP."
+msgid "Add support for UPNP."
 msgstr "Añadir soporte para UPNP. "
 
 #: src/OptionsWindows.cc:59
@@ -810,7 +810,7 @@
 msgstr "Añadir soporte para buscar nodos en la red local. "
 
 #: src/OptionsWindows.cc:60
-msgid "Adds suppport for exchanging metadata from peers. (*)"
+msgid "Adds support for exchanging metadata from peers. (*)"
 msgstr "Añadir soporte para intercambiar metadatos entre los clientes. (*)"
 
 #: src/OptionsWindows.cc:61
diff -Nur lince-1.3/po/it.po lince/po/it.po
--- lince-1.3/po/it.po	2012-11-02 18:07:56.589271145 +0100
+++ lince/po/it.po	2012-11-02 18:06:57.220493000 +0100
@@ -906,11 +906,11 @@
 msgstr "a"
 
 #: src/OptionsWindows.cc:59
-msgid "Add suppport for DHT. (*)"
+msgid "Add support for DHT. (*)"
 msgstr "Aggiungi supporto per DHT. (*)"
 
 #: src/OptionsWindows.cc:60
-msgid "Add suppport for UPNP. (*)"
+msgid "Add support for UPNP. (*)"
 msgstr "Aggiungi supporto per UPNP. (*)"
 
 #: src/OptionsWindows.cc:61
@@ -918,7 +918,7 @@
 msgstr "Aggiungi supporto per la ricerca di nodi nella rete locale. (*)"
 
 #: src/OptionsWindows.cc:62
-msgid "Adds suppport for exchanging metadata from peers. (*)"
+msgid "Adds support for exchanging metadata from peers. (*)"
 msgstr "Aggiungi supporto per lo scambio di metadati tra nodi. (*)"
 
 #: src/OptionsWindows.cc:63
diff -Nur lince-1.3/src/ClientsInfoNoteBookTab.cc lince/src/ClientsInfoNoteBookTab.cc
--- lince-1.3/src/ClientsInfoNoteBookTab.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/ClientsInfoNoteBookTab.cc	2012-11-02 18:06:57.220493000 +0100
@@ -34,6 +34,8 @@
 #include "Lince/SessionManager.h"
 #include "Lince/Utils.h"
 
+#include "libtorrent/peer_info.hpp"
+
 using namespace Lince;
 
 ClientsInfoNoteBookTab::ClientsInfoNoteBookTab(TorrentInfoTab *tit):
diff -Nur lince-1.3/src/FilesInfoNoteBookTab.cc lince/src/FilesInfoNoteBookTab.cc
--- lince-1.3/src/FilesInfoNoteBookTab.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/FilesInfoNoteBookTab.cc	2012-11-02 18:06:57.220493000 +0100
@@ -169,7 +169,7 @@
 {
 	int i;
 	libtorrent::torrent_info t_info = t->get_torrent_info();
-	std::vector<libtorrent::file_entry>::const_iterator file_iterator;
+	libtorrent::torrent_info::file_iterator file_iterator;
 	Gtk::TreeModel::Row row;
 	Gtk::TreeModel::iterator row_iter;
 	libtorrent::file_entry file_entry;
@@ -188,7 +188,8 @@
 
 			row[filesColumns.m_file_firstpiece] = first;
 			row[filesColumns.m_file_lastpiece] = last;
-			row[filesColumns.m_file_name] = file_iterator->path.leaf();
+			//row[filesColumns.m_file_name] = file_iterator->path.filename().native();
+			row[filesColumns.m_file_name] = file_entry.path;
 			row[filesColumns.m_file_size] = size_to_string(file_entry.size);
 			row[filesColumns.file_size] = file_entry.size;
 			row[filesColumns.m_file_percentage] = 0;
diff -Nur lince-1.3/src/GeneralInfoNoteBookTab.cc lince/src/GeneralInfoNoteBookTab.cc
--- lince-1.3/src/GeneralInfoNoteBookTab.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/GeneralInfoNoteBookTab.cc	2012-11-02 18:06:57.220493000 +0100
@@ -40,6 +40,7 @@
 
 GraphicBar::GraphicBar()
 {
+	percentage = 0.0;
 	currentPieces = 0;
 	totalPieces = 1;
 	percentageMode = false;
@@ -531,7 +532,7 @@
 		storeValueLabel.set_label((Glib::build_filename(
 					t->save_path().native_directory_string(), t->name())));*/
 
-	storeValueLabel.set_label(t->save_path().native_file_string());
+	storeValueLabel.set_label(t->save_path());
 	
 	tempValue << t->get_hash();
 	hashValueLabel.set_label(tempValue.str());
diff -Nur lince-1.3/src/InfoNoteBook.cc lince/src/InfoNoteBook.cc
--- lince-1.3/src/InfoNoteBook.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/InfoNoteBook.cc	2012-11-02 18:06:57.220493000 +0100
@@ -52,6 +52,7 @@
 	append_page(*optionsTab.getTabChild(), optionsTab.getNameTab());
 	
 	updatingData = false;
+	last_page = 0;
 }
 
 void InfoNoteBook::updateInfoNoteBook(Lince::Torrent *t){
diff -Nur lince-1.3/src/LoadOneTorrentDialog.cc lince/src/LoadOneTorrentDialog.cc
--- lince-1.3/src/LoadOneTorrentDialog.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/LoadOneTorrentDialog.cc	2012-11-02 18:06:57.220493000 +0100
@@ -177,15 +177,20 @@
 		// Fill torrent's data
 		torrentname = t.name();
 		priorities->resize(t.num_files(), 1);
-		for (int i=0; i<t.num_files(); i++)
-			newfilenames->push_back(t.file_at(i).path.string());
+		for (int i=0; i<t.num_files(); i++){
+			newfilenames->push_back(t.file_at(i).path);
+			Glib::ustring fileName(t.file_at(i).path);
+			addRow(pos, fileName, t.file_at(i).size);
+			pos++;
+		}
 
 		torrentName->set_label(torrentname);
-		for (libtorrent::torrent_info::file_iterator i = t.begin_files(); i != t.end_files(); ++i){
+		// This has become obsolete in libtorrent 0.16.x
+		/*for (libtorrent::torrent_info::file_iterator i = t.begin_files(); i != t.end_files(); ++i){
 			Glib::ustring fileName(i->path.string());
 			addRow(pos, fileName, i->size);
 			pos++;
-		}
+		}*/
 	}
 	catch (libtorrent::invalid_torrent_file&){
 	}
diff -Nur lince-1.3/src/LoadTorrentDialog.cc lince/src/LoadTorrentDialog.cc
--- lince-1.3/src/LoadTorrentDialog.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/LoadTorrentDialog.cc	2012-11-02 18:06:57.220493000 +0100
@@ -352,7 +352,7 @@
 		td.priorities.resize(t.num_files(), 1);
 
 		for (int i=0; i<t.num_files(); i++){
-			td.newfilenames.push_back(t.file_at(i).path.string());
+			td.newfilenames.push_back(t.file_at(i).path);
 			td.sizes.push_back(t.file_at(i).size);
 		}
 	}
@@ -410,11 +410,11 @@
 			addTorrentsNameRow(torrentsPaths[i]);
 			torrentLoaded = true;
 		}
-		catch (libtorrent::invalid_encoding)
+		catch (const libtorrent::invalid_encoding &)
 		{
 			Utils::warningDialog(_("Selected file is not a torrent"), torrentsPaths[i]);
 		}
-		catch (libtorrent::invalid_torrent_file)
+		catch (const libtorrent::invalid_torrent_file &)
 		{
 			Utils::warningDialog(_("Selected file is not a torrent"), torrentsPaths[i]);
 		}
@@ -489,11 +489,11 @@
 		addTorrentsNameRow(url);
 		torrentsNamesTreeSelection->select(--(torrentsNamesTreeModel->children().end()));
 	}
-	catch (libtorrent::invalid_encoding)
+	catch (const libtorrent::invalid_encoding &)
 	{
 		Utils::warningDialog(_("Selected file is not a torrent"), url);
 	}
-	catch (libtorrent::invalid_torrent_file)
+	catch (const libtorrent::invalid_torrent_file &)
 	{
 		Utils::warningDialog(_("Selected file is not a torrent"), url);
 	}
diff -Nur lince-1.3/src/LogTab.cc lince/src/LogTab.cc
--- lince-1.3/src/LogTab.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/LogTab.cc	2012-11-02 18:06:57.220493000 +0100
@@ -103,7 +103,7 @@
 	appendBoldText(std::string(PACKAGE) + std::string(_(" started\n")));
 
 	// Open Log File
-	fileName = Glib::build_filename(get_base_directory().native_directory_string(),
+	fileName = Glib::build_filename(get_base_directory().native(),
 									std::string("session.log"));
 
 	fileStream.open(std::string(fileName.c_str()));
diff -Nur lince-1.3/src/main.cc lince/src/main.cc
--- lince-1.3/src/main.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/main.cc	2012-11-02 18:06:57.220493000 +0100
@@ -196,9 +196,6 @@
 	bind_textdomain_codeset(GETTEXT_PACKAGE, "UTF-8");
 	textdomain(GETTEXT_PACKAGE);
 	
-	/* Support for native paths */
-	boost::filesystem::path::default_name_check(boost::filesystem::native);
-	
 	Gtk::Main kit(ac, av);
 	
 	if(!Glib::thread_supported()) 
diff -Nur lince-1.3/src/MainWindow.cc lince/src/MainWindow.cc
--- lince-1.3/src/MainWindow.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/MainWindow.cc	2012-11-02 18:06:57.220493000 +0100
@@ -343,8 +343,8 @@
 			return;
 		}
 
-		torrentsavedest = Glib::build_filename(get_base_directory().native_directory_string()
-											   , torrentpath.leaf());
+		torrentsavedest = Glib::build_filename(get_base_directory().native()
+											   , torrentpath.filename().native());
 
 		// Copy torrent to desired place only if desired place and load place are not the same place
 		if (guiSetupManager.getBool(GuiSetupManager::USE_DOWNLOAD_TORRENTS_DIRECTORY)
@@ -356,7 +356,7 @@
 				{
 				boost::filesystem::copy_file(torrentpath, boost::filesystem::path(
 					Glib::build_filename(guiSetupManager.get(GuiSetupManager::DOWNLOAD_TORRENTS_DIRECTORY)
-					, torrentpath.leaf())));
+					, torrentpath.filename().native())));
 				}
 				catch(std::exception &)
 				{
@@ -366,7 +366,7 @@
 				{
 					boost::filesystem::remove(torrentpath);
 					torrentpath = Glib::build_filename(guiSetupManager.get(
-						GuiSetupManager::DOWNLOAD_TORRENTS_DIRECTORY), torrentpath.leaf());
+						GuiSetupManager::DOWNLOAD_TORRENTS_DIRECTORY), torrentpath.filename().native());
 				}
 			}
 		}
diff -Nur lince-1.3/src/MakeTorrent.cc lince/src/MakeTorrent.cc
--- lince-1.3/src/MakeTorrent.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/MakeTorrent.cc	2012-11-02 18:06:57.220493000 +0100
@@ -179,7 +179,7 @@
 void MakeTorrent::makeTorrent(Glib::ustring torrentFileName){
 	libtorrent::file_storage fs;
 	
-	libtorrent::add_files(fs, path);
+	libtorrent::add_files(fs, path.native());
 	
 	libtorrent::create_torrent t(fs);
 	
@@ -190,7 +190,7 @@
 	
 	pulse = double(1/(double) fs.num_pieces());
 	
-	libtorrent::set_piece_hashes(t, path.branch_path(), 
+	libtorrent::set_piece_hashes(t, path.branch_path().native(),
 				boost::bind(&MakeTorrent::launchUpdateProgressBar, this, _1));
 	
 	Lince::save_entry(torrentFileName.c_str(), t.generate());
@@ -210,8 +210,8 @@
 	
 	if (file == "") return;
 	
-	path = boost::filesystem::complete(file.c_str());
-	addFiles(path.branch_path(), path.leaf());
+	path = boost::filesystem::path(file.c_str());
+	addFiles(path.branch_path(), path.filename());
 }
 
 void MakeTorrent::addDirectory(){
@@ -221,19 +221,19 @@
 	
 	if (directory == "") return;
 	
-	path = boost::filesystem::complete(directory.c_str());
-	addFiles(path.branch_path(), path.leaf());
+	path = boost::filesystem::path(directory.c_str());
+	addFiles(path.branch_path(), path.filename());
 }
 
 void MakeTorrent::addFiles(boost::filesystem::path const& p, boost::filesystem::path const& l)
 {
-	if (l.leaf()[0] == '.') return;
+	if (l.filename().native()[0] == '.') return;
 	boost::filesystem::path f(p / l);
 	
 	if (boost::filesystem::is_directory(f)){
 		for (boost::filesystem::directory_iterator i(f), end; i != end; ++i){
 			try{
-				addFiles(p, l / i->filename());
+				addFiles(p, l / i->path().filename());
 			}
 			catch(std::exception &)
 			{
diff -Nur lince-1.3/src/NoteBookManager.h lince/src/NoteBookManager.h
--- lince-1.3/src/NoteBookManager.h	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/NoteBookManager.h	2012-11-02 18:06:57.220493000 +0100
@@ -106,7 +106,7 @@
 		int getTorrentsTabPaneSeparatorPosition() { return TT.getPaneSeparatorPosition(); };
 		int getDownloadingSelectedRow() { return TT.getDownloadingSelectedRow(); };
 		int getDownloadedSelectedRow() { return TT.getDownloadedSelectedRow(); };
-		int getDownloadingSelectedTorrents(void) { TT.getDownloadingSelectedTorrents(); };
+		int getDownloadingSelectedTorrents(void) { return TT.getDownloadingSelectedTorrents(); };
 		Lince::torrentPosition getSelectedTorrentPosition(void) { return TT.getSelectedTorrentPosition(); };
 		std::vector<Lince::torrentPosition> getSelectedTorrentsPositions(void) { return TT.getSelectedTorrentsPositions(); };
 
diff -Nur lince-1.3/src/OptionsWindows.cc lince/src/OptionsWindows.cc
--- lince-1.3/src/OptionsWindows.cc	2012-11-02 18:07:56.589271145 +0100
+++ lince/src/OptionsWindows.cc	2012-11-02 18:06:57.220493000 +0100
@@ -54,10 +54,10 @@
 	tcpText1(_("Port range:")),
 	tcpText2(_("to")),
 	tcpRow(false, 5),
-	dhtActivateButton(_("Add suppport for DHT.")),
-	upnpActivateButton(_("Add suppport for UPNP.")),
+	dhtActivateButton(_("Add support for DHT.")),
+	upnpActivateButton(_("Add support for UPNP.")),
 	localDiscoveryActivateButton(_("Add support for searching peers in the local network.")),
-	metadataActivateButton(_("Adds suppport for exchanging metadata from peers. (*)")),
+	metadataActivateButton(_("Adds support for exchanging metadata from peers. (*)")),
 	pexActivateButton(_("Adds support for uTorrent peer exchange. (*)")),
 	smartBanButton(_("Adds support for smart ban plugin. (*)")),
 	advancedBindToInterfaceLabel(_("(*) Bind to interface:")),
diff -Nur lince-1.3/src/TorrentInfoTab.cc lince/src/TorrentInfoTab.cc
--- lince-1.3/src/TorrentInfoTab.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/TorrentInfoTab.cc	2012-11-02 18:06:57.220493000 +0100
@@ -112,7 +112,7 @@
 	Lince::Torrent *t = tm->get_torrent_at(tp);
 
 	Glib::ustring save_directory = Utils::getDirectoryFromDialog(_("Select torrent save path"),
-			t->save_path().native_directory_string());
+			t->save_path());
 	if (save_directory.size())
 		t->move_storage(save_directory.c_str());
 
diff -Nur lince-1.3/src/TorrentsWindow.cc lince/src/TorrentsWindow.cc
--- lince-1.3/src/TorrentsWindow.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/TorrentsWindow.cc	2012-11-02 18:06:57.220493000 +0100
@@ -497,12 +497,12 @@
 
 	if (torrentsWindowType == DOWNLOADING_TORRENTS)
 	{
-		fname = Glib::build_filename(get_base_directory().native_directory_string(),
+		fname = Glib::build_filename(get_base_directory().native(),
 									 Glib::ustring("downloadcolumns.xml"));
 	}
 	else
 	{
-		fname = Glib::build_filename(get_base_directory().native_directory_string(),
+		fname = Glib::build_filename(get_base_directory().native(),
 									 Glib::ustring("finishedcolumns.xml"));
 	}
 
@@ -563,10 +563,10 @@
 	Glib::ustring fname;
 
 	if (torrentsWindowType == DOWNLOADING_TORRENTS)
-		fname = Glib::build_filename(get_base_directory().native_directory_string(),
+		fname = Glib::build_filename(get_base_directory().native(),
 									 Glib::ustring("downloadcolumns.xml"));
 	else
-		fname = Glib::build_filename(get_base_directory().native_directory_string(),
+		fname = Glib::build_filename(get_base_directory().native(),
 									 std::string("finishedcolumns.xml"));
 
 	writer.setFileName(fname);
@@ -677,7 +677,7 @@
 	int i;
 	Gtk::TreeModel::Row row;
 	Gtk::TreeModel::iterator j;
-
+	
 	TorrentsManagerPtr tm;
 	tm = Core::get_torrents_manager();
 	Torrent *t;
diff -Nur lince-1.3/src/Utils.cc lince/src/Utils.cc
--- lince-1.3/src/Utils.cc	2012-06-28 13:50:14.000000000 +0200
+++ lince/src/Utils.cc	2012-11-02 18:06:57.220493000 +0100
@@ -172,15 +172,15 @@
 			{
 				ssm->add_torrent_to_session(tp, td, prio, nfilenam, paused);
 			}
-			catch(libtorrent::duplicate_torrent)
+			catch(const libtorrent::duplicate_torrent &)
 			{
 				warningDialog(_("Torrent is already being downloaded"),
-						tp.filename());
+						tp.filename().native());
 			}
-			catch(Lince::queue_limit_reached)
+			catch(const Lince::queue_limit_reached &)
 			{
 				warningDialog(_("The queue limit has been reached."),
-						tp.filename());
+						tp.filename().native());
 			}
 		}
 		else
@@ -211,15 +211,15 @@
 			{
 				ssm->add_magnet_to_session(magnet, savepath);
 			}
-			catch(libtorrent::duplicate_torrent)
+			catch(const libtorrent::duplicate_torrent &)
 			{
 				warningDialog(_("Magnet link is already being downloaded"), sha1);
 			}
-			catch(Lince::queue_limit_reached)
+			catch(const Lince::queue_limit_reached &)
 			{
 				warningDialog(_("The queue limit has been reached."), sha1);
 			}
-			catch(Lince::metadata_not_activated)
+			catch(const Lince::metadata_not_activated &)
 			{
 				warningDialog(_("Exchange metadata between clients option is not activated."), sha1);
 			}
@@ -460,14 +460,14 @@
 
 	bool isATorrentFile(const boost::filesystem::path &tp){
 		try{
-			libtorrent::torrent_info t(tp);
+			libtorrent::torrent_info t(tp.native());
 			if (!t.num_files())
 				return false;
 		}
-		catch (libtorrent::invalid_torrent_file){
+		catch (const libtorrent::invalid_torrent_file &){
 			return false;
 		}
-		catch (libtorrent::invalid_encoding){
+		catch (const libtorrent::invalid_encoding &){
 			return false;
 		}
 
