Index: domotiga-0.1.192/DomotiGa.desktop
===================================================================
--- domotiga-0.1.192.orig/DomotiGa.desktop	2010-12-24 00:23:50.509492003 +0100
+++ domotiga-0.1.192/DomotiGa.desktop	2010-12-24 00:24:49.009492000 +0100
@@ -1,11 +1,8 @@
-
 [Desktop Entry]
-Encoding=UTF-8
 Version=1.0
 Type=Application
 Terminal=false
-Name[en_US]=DomotiGa
-Exec=/home/ron/domotiga/DomotiGa.gambas
-Icon[en_US]=/home/ron/domotiga/icons/logo.png
+Exec=domotiga
 Name=DomotiGa
-Icon=/home/ron/domotiga/icons/logo.png
+Icon=domotiga
+Categories=Utility;
Index: domotiga-0.1.192/DomotiGaServer.desktop
===================================================================
--- domotiga-0.1.192.orig/DomotiGaServer.desktop	2010-12-24 00:24:58.789492000 +0100
+++ domotiga-0.1.192/DomotiGaServer.desktop	2010-12-24 00:25:30.229492001 +0100
@@ -1,10 +1,8 @@
 [Desktop Entry]
-Comment=
-Encoding=UTF-8
-Exec='/home/ron/domotiga/DomotiGaServer.gambas'
-Icon=/home/ron/domotiga/DomotiGaServer/domotiga.png
+Exec=domotiga-server
+Icon=domotiga
 Name=DomotiGaServer
 StartupNotify=true
 Terminal=true
-TerminalOptions=
 Type=Application
+Categories=Utility;
