Index: calibre-2.69.0/recipes/lenta_ru.recipe
===================================================================
--- calibre-2.69.0.orig/recipes/lenta_ru.recipe	2016-10-09 22:09:02.146459672 +0200
+++ calibre-2.69.0/recipes/lenta_ru.recipe	2016-10-09 22:09:46.435406359 +0200
@@ -4,11 +4,13 @@
 Lenta.ru
 '''
 
-from calibre.web.feeds.feedparser import parse
 from calibre.ebooks.BeautifulSoup import Tag
 from calibre.web.feeds.news import BasicNewsRecipe
+from feedparser import parse
+from functools import partial
 import re
 
+parse = partial(parse, agent='Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US; rv:1.9.2.11) Gecko/20101012 Firefox/3.6.11')
 
 class LentaRURecipe(BasicNewsRecipe):
     title = u'Lenta.ru: \u041d\u043e\u0432\u043e\u0441\u0442\u0438'
Index: calibre-2.69.0/src/calibre/web/feeds/__init__.py
===================================================================
--- calibre-2.69.0.orig/src/calibre/web/feeds/__init__.py	2016-10-09 22:09:02.146459672 +0200
+++ calibre-2.69.0/src/calibre/web/feeds/__init__.py	2016-10-09 22:10:03.531770787 +0200
@@ -12,6 +12,11 @@
 from calibre.utils.date import dt_factory, utcnow, local_tz
 from calibre.utils.cleantext import clean_ascii_chars, clean_xml_chars
 
+from feedparser import parse
+from functools import partial
+
+parse = partial(parse, agent='Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US; rv:1.9.2.11) Gecko/20101012 Firefox/3.6.11')
+
 class Article(object):
 
     def __init__(self, id, title, url, author, summary, published, content):
@@ -331,7 +336,6 @@
                   max_articles_per_feed=100,
                   get_article_url=lambda item: item.get('link', None),
                   log=default_log):
-    from calibre.web.feeds.feedparser import parse
     # Handle unclosed escaped entities. They trip up feedparser and HBR for one
     # generates them
     raw_xml = re.sub(r'(&amp;#\d+)([^0-9;])', r'\1;\2', raw_xml)
