# set install path of the manual
Index: bibfilex-1.2.8/unit1.pas
===================================================================
--- bibfilex-1.2.8.orig/unit1.pas	2016-11-06 03:13:01.318896763 +0100
+++ bibfilex-1.2.8/unit1.pas	2016-11-06 03:14:02.768180392 +0100
@@ -2749,8 +2749,7 @@
 procedure TfmMain.miShowManualClick(Sender: TObject);
 begin
   // Show manual
-  if OpenDocument(ExtractFileDir(Application.ExeName) + DirectorySeparator +
-    'bibfilex-manual.pdf') = False then
+  if OpenDocument('/usr/share/bibfilex/bibfilex-manual.pdf') = False then
   begin
     MessageDlg('The user manual is not installed; ' +
       'download it from Bibfilex website, ' +
