# fix include path
Index: pragha-1.3.3/plugins/lastfm/pragha-lastfm-plugin.c
===================================================================
--- pragha-1.3.3.orig/plugins/lastfm/pragha-lastfm-plugin.c	2015-09-24 01:06:59.000000000 +0200
+++ pragha-1.3.3/plugins/lastfm/pragha-lastfm-plugin.c	2015-09-24 20:04:12.158790792 +0200
@@ -30,7 +30,7 @@
 #include <gmodule.h>
 #include <gtk/gtk.h>
 
-#include <cdio/paranoia/cdda.h>
+#include <cdio/paranoia.h>
 #include <cdio/cd_types.h>
 #include <cddb/cddb.h>
 
