Do not check for updates.
Index: dispcalgui-0.9.9.1/dispcalGUI/dispcalGUI.py
===================================================================
--- dispcalgui-0.9.9.1.orig/dispcalGUI/dispcalGUI.py	2012-04-01 16:03:23.000000000 +0200
+++ dispcalgui-0.9.9.1/dispcalGUI/dispcalGUI.py	2012-05-06 11:41:22.792713131 +0200
@@ -1278,8 +1278,8 @@
 			wx.GetApp().progress_dlg.Hide()
 		
 		# Check for updates if configured
-		if getcfg("update_check"):
-			self.app_update_check_handler(None, silent=True)
+		#if getcfg("update_check"):
+		#	self.app_update_check_handler(None, silent=True)
 	
 	def log(self):
 		""" Put log buffer contents into the log window. """
