# add missing includes and compile with position independent code
Index: drawpile-0.7.1/CMakeLists.txt
===================================================================
--- drawpile-0.7.1.orig/CMakeLists.txt	2013-12-08 15:35:20.030677946 +0100
+++ drawpile-0.7.1/CMakeLists.txt	2013-12-08 15:35:20.026677946 +0100
@@ -48,7 +48,7 @@
 find_package ( ZLIB REQUIRED )
 
 # Enable C++11
-list(APPEND CMAKE_CXX_FLAGS "-std=c++11")
+list(APPEND CMAKE_CXX_FLAGS "-std=c++11 -fPIC -I/usr/include/qt5/QtNetwork/ -I/usr/include/qt5 -I/usr/include/qt5/QtCore/")
 # GCC <4.7 may need this instead:
 #list(APPEND CMAKE_CXX_FLAGS "-std=c++0x")
 
