# Remove dependency on Qt5.1
Index: drawpile-0.7.1/src/client/canvasview.cpp
===================================================================
--- drawpile-0.7.1.orig/src/client/canvasview.cpp	2013-12-05 21:13:02.000000000 +0100
+++ drawpile-0.7.1/src/client/canvasview.cpp	2013-12-08 15:50:50.486670002 +0100
@@ -18,7 +18,6 @@
    Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 */
 
-#include <QtMath>
 #include <QMouseEvent>
 #include <QTabletEvent>
 #include <QScrollBar>
@@ -420,6 +419,11 @@
 	sceneChanged();
 }
 
+inline float qRadiansToDegrees(float radians)
+{
+    return radians * float(180/M_PI);
+}
+
 /**
  * @param x x coordinate
  * @param y y coordinate
@@ -430,8 +434,8 @@
 	const int dy = _dragy - y;
 
 	if(_isdragging==ROTATE) {
-		qreal preva = qAtan2( width()/2 - _dragx, height()/2 - _dragy );
-		qreal a = qAtan2( width()/2 - x, height()/2 - y );
+		qreal preva = atan2( width()/2 - _dragx, height()/2 - _dragy );
+		qreal a = atan2( width()/2 - x, height()/2 - y );
 		setRotation(rotation() + qRadiansToDegrees(preva-a));
 	} else {
 		QScrollBar *ver = verticalScrollBar();
Index: drawpile-0.7.1/src/client/textloader.cpp
===================================================================
--- drawpile-0.7.1.orig/src/client/textloader.cpp	2013-12-05 21:13:02.000000000 +0100
+++ drawpile-0.7.1/src/client/textloader.cpp	2013-12-08 17:16:12.870626270 +0100
@@ -85,7 +85,7 @@
 	if(str.at(0) == '#')
 	{
 		bool ok;
-		uint i = str.midRef(1).toUInt(&ok, 16);
+		uint i = str.mid(1).toUInt(&ok, 16);
 		if(!ok)
 			throw SyntaxError(QString("'%1'' is not an integer!").arg(str));
 		return i;
Index: drawpile-0.7.1/src/client/ora/qzip.cpp
===================================================================
--- drawpile-0.7.1.orig/src/client/ora/qzip.cpp	2013-12-05 21:13:02.000000000 +0100
+++ drawpile-0.7.1/src/client/ora/qzip.cpp	2013-12-08 18:19:35.362593806 +0100
@@ -44,7 +44,8 @@
 #include "zipreader.h"
 #include "zipwriter.h"
 #include <qdatetime.h>
-#include <qplatformdefs.h>
+#include <fcntl.h>
+#include <sys/stat.h>
 #include <qendian.h>
 #include <qdebug.h>
 #include <qdir.h>
Index: drawpile-0.7.1/src/client/widgets/brushpreview.cpp
===================================================================
--- drawpile-0.7.1.orig/src/client/widgets/brushpreview.cpp	2013-12-05 21:13:02.000000000 +0100
+++ drawpile-0.7.1/src/client/widgets/brushpreview.cpp	2013-12-08 18:22:00.878592564 +0100
@@ -18,7 +18,6 @@
    Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 */
 #include <QDebug>
-#include <QtMath>
 #include <QPaintEvent>
 #include <QPainter>
 #include <QEvent>
@@ -136,7 +135,7 @@
 
 			const qreal fx = x/qreal(strokew);
 			const qreal pressure = qBound(0.0, ((fx*fx) - (fx*fx*fx))*6.756, 1.0);
-			const int y = qRound(qSin(phase) * strokeh);
+			const int y = qRound(sin(phase) * strokeh);
 			layer->drawLine(0, brush_,
 					dpcore::Point(offx+lastx,offy+lasty, lastp),
 					dpcore::Point(offx+x, offy+y, pressure), distance);
Index: drawpile-0.7.1/doc/CMakeLists.txt
===================================================================
--- drawpile-0.7.1.orig/doc/CMakeLists.txt	2013-12-05 21:13:02.000000000 +0100
+++ drawpile-0.7.1/doc/CMakeLists.txt	2013-12-08 18:25:20.778590857 +0100
@@ -3,7 +3,6 @@
 	../COPYING
 	../README.md
 	openraster.txt
-	textloader.txt
 )
 
 SET(man1
