Fix lua version.
Index: qosmic-1.5.0/qosmic.pro
===================================================================
--- qosmic-1.5.0.orig/qosmic.pro	2011-11-10 19:10:23.810564777 +0100
+++ qosmic-1.5.0/qosmic.pro	2011-11-10 19:10:40.286564288 +0100
@@ -49,7 +49,7 @@
 
 link_pkgconfig {
 	message("Config using pkg-config version "$$system(pkg-config --version))
-	PKGCONFIG = flam3 lua
+	PKGCONFIG = flam3 lua5.1
 
 	## The directory that contains flam3-palettes.xml must be set here.  If
 	## your system has pkg-config, this should find the flam3 palettes.
