# do not access the debian directory with cmake.
Index: qstopmotion-2.0.0/CMakeLists.txt
===================================================================
--- qstopmotion-2.0.0.orig/CMakeLists.txt	2015-01-02 22:11:37.031796232 +0100
+++ qstopmotion-2.0.0/CMakeLists.txt	2015-01-02 22:13:34.699792732 +0100
@@ -319,12 +319,6 @@
 #     qstopmotion.1         # Linux man page
 )
 
-set(DEBIAN_FILES
-    debian/changelog             # changelog file
-    debian/copyright             # copyright file
-    debian/qstopmotion.menu      # .menu is installed in a separate step for Debian systems
-)
-
 ###############################################################################
 # GStreamer support
 #
@@ -565,8 +559,6 @@
     file(COPY ${ICON_FILES} DESTINATION "share/qstopmotion/")
     file(COPY ${GENERAL_FILES} DESTINATION "share/qstopmotion/")
     file(COPY ${UNIX_FILES} DESTINATION "share/qstopmotion/")
-    file(COPY ${DEBIAN_FILES} DESTINATION "share/qstopmotion/debian/")
-    file(COPY ${FEDORA_FILES} DESTINATION "share/qstopmotion/rpm/")
 
 ENDIF(WIN32)
 
@@ -680,22 +672,6 @@
     )
 ENDIF(UNIX)
 
-# installation on DEBIAN systems
-IF(CPACK_BINARY_DEB)
-    message("Special task for Debian package!")
-    # installation of the icon file
-    install(
-        FILES "debian/qstopmotion.xpm"
-        DESTINATION ${QSTOPMOTION_PIXMAP_INSTALL_DIR}
-    )
-
-    # installation of the .menufile file
-    install(
-        FILES "debian/qstopmotion.menu"
-        DESTINATION ${QSTOPMOTION_MENUFILE_INSTALL_DIR}
-    )
-ENDIF(CPACK_BINARY_DEB)
-
 # installation on Windows systems
 IF(WIN32)
     # installation of ffmpeg
@@ -811,30 +787,6 @@
 # SET(CPACK_NSIS_MUI_FINISHPAGE_RUN "qstopmotion.exe"
 
 ###############################################################################
-# DEB package - Linux only (Debian, OpenSuse, Ubuntu, ...)
-###############################################################################
-
-# Mandatory
-# CPACK_DEBIAN_PACKAGE_NAME = CPACK_PACKAGE_NAME
-# CPACK_DEBIAN_PACKAGE_VERSION = CPACK_PACKAGE_VERSION
-# CPACK_DEBIAN_PACKAGE_ARCHITECTURE = dpkg output       # dpkg --print-architecture ???
-# CPACK_DEBIAN_PACKAGE_MAINTAINER = CPACK_PACKAGE_CONTACT
-# CPACK_DEBIAN_PACKAGE_DESCRIPTION = CPACK_PACKAGE_DESCRIPTION_SUMMARY
-SET(CPACK_DEBIAN_PACKAGE_SECTION "graphics")
-SET(CPACK_DEBIAN_PACKAGE_PRIORITY "optional")
-
-
-# Options
-SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libc6 (>=2.11), ffmpeg (>=0.8.9), gstreamer-tools (>=0.10.36), libqt4-core (>=4.8.2), libqt4-gui (>=4.8.2), libqt4-xml (>=4.8.2)")
-SET(CPACK_DEBIAN_PACKAGE_HOMEPAGE ${QSTOPMOTION_HOMEPAGE})
-# CPACK_DEBIAN_PACKAGE_SHLIBDEPS
-# SET(CPACK_DEBIAN_PACKAGE_DEBUG "ON")
-# CPACK_DEBIAN_PACKAGE_DEBUG
-# CPACK_DEBIAN_PACKAGE_ENHANCES
-# CPACK_DEBIAN_PACKAGE_BREAKS
-# CPACK_DEBIAN_PACKAGE_CONFLICTS
-# CPACK_DEBIAN_PACKAGE_PROVIDES
-# CPACK_DEBIAN_PACKAGE_REPLACES
 
 ###############################################################################
 # RPM package - Linux only (Fedora, Red Hat, CentOS, Mandriva, ...)
