# do not access the debian directory with cmake.
Index: qstopmotion-2.0.2/CMakeLists.txt
===================================================================
--- qstopmotion-2.0.2.orig/CMakeLists.txt	2015-09-15 19:46:36.000000000 +0200
+++ qstopmotion-2.0.2/CMakeLists.txt	2015-09-23 19:23:40.108945000 +0200
@@ -333,21 +333,6 @@
     linux/qstopmotion.1.gz      # Linux man page compressed
 )
 
-SET(
-    DEB_FILES
-    deb/qstopmotion.xpm       # DEB Icon is installed in a separate step
-    deb/changelog             # DEB changelog file
-    deb/changelog.gz          # DEB changelog file compressed
-    deb/copyright             # DEB copyright file
-    deb/qstopmotion.menu      # DEB .menu is installed in a separate step for Debian systems
-    deb/postinst              # DEB post installation script file, call update-menus
-)
-
-SET(
-    RPM_FILES
-    rpm/qstopmotion.png       # RPM Icon is installed in a separate step
-)
-
 ###############################################################################
 # GStreamer support
 #
@@ -586,8 +571,6 @@
     file(COPY ${ICON_FILES} DESTINATION "share/qstopmotion/")
     file(COPY ${GENERAL_FILES} DESTINATION "share/qstopmotion/")
     file(COPY ${UNIX_FILES} DESTINATION "share/qstopmotion/")
-    file(COPY ${DEB_FILES} DESTINATION "share/qstopmotion/deb/")
-    file(COPY ${RPM_FILES} DESTINATION "share/qstopmotion/rpm/")
 
 ENDIF(WIN32)
 
@@ -707,52 +690,6 @@
     )
 ENDIF(UNIX)
 
-# installation on DEB systems
-IF(CPACK_BINARY_DEB)
-    message("Special task for deb package!")
-    
-    # installation of the icon file
-    install(
-        FILES "deb/qstopmotion.xpm"
-        DESTINATION ${QSTOPMOTION_PIXMAP_INSTALL_DIR}
-    )
-
-    # installation of the .menufile file
-    install(
-        FILES "deb/qstopmotion.menu"
-        DESTINATION ${QSTOPMOTION_MENUFILE_INSTALL_DIR}
-    )
-
-    # installation of the changelog file
-    install(
-        FILES "deb/changelog.gz"
-        DESTINATION ${QSTOPMOTION_MANUAL_INSTALL_DIR}
-    )
-
-    # installation of the copyright file
-    install(
-        FILES "deb/copyright"
-        DESTINATION ${QSTOPMOTION_MANUAL_INSTALL_DIR}
-    )
-ENDIF(CPACK_BINARY_DEB)
-
-# installation on RPM systems
-IF(CPACK_BINARY_RPM)
-    message("Special task for rpm package!")
-    
-    # installation of the general-purpose files
-    install(
-        FILES ${GENERAL_FILES}
-        DESTINATION ${QSTOPMOTION_GENERALS_INSTALL_DIR}
-    )
-
-    # installation of the icon file
-    install(
-        FILES "rpm/qstopmotion.png"
-        DESTINATION ${QSTOPMOTION_PIXMAP_INSTALL_DIR}
-    )
-ENDIF(CPACK_BINARY_RPM)
-
 # installation on Windows systems
 IF(WIN32)
     # installation of ffmpeg
@@ -965,171 +902,6 @@
 # SET(CPACK_NSIS_MUI_FINISHPAGE_RUN "qstopmotion.exe"
 
 ###############################################################################
-# DEB package - Linux only (Debian, OpenSuse, Ubuntu, ...)
-###############################################################################
-
-IF ("${LSB_DISTRIB}" MATCHES "Ubuntu|Debian")
-    # We need to alter the architecture names as per distro rules
-    IF ("${CPACK_PACKAGE_ARCHITECTURE}" MATCHES "i[3-6]86")
-        SET(
-            CPACK_PACKAGE_ARCHITECTURE
-            "i386"
-        )
-    ENDIF ("${CPACK_PACKAGE_ARCHITECTURE}" MATCHES "i[3-6]86")
-    IF ("${CPACK_PACKAGE_ARCHITECTURE}" MATCHES "x86_64")
-        SET(
-            CPACK_PACKAGE_ARCHITECTURE
-            "amd64"
-        )
-    ENDIF ("${CPACK_PACKAGE_ARCHITECTURE}" MATCHES "x86_64")
-    SET(
-        CPACK_SYSTEM_NAME
-        "${LSB_DISTRIB}-${CPACK_PACKAGE_ARCHITECTURE}"
-    )
-
-    # Set the dependencies based on the distro version
-    # thus only one package is build you must list here *any* depending libraries,
-    # even if they are only used in one module
-    IF ("${LSB_DISTRIB}" MATCHES "Ubuntu12.04")
-        SET(
-            CPACK_DEBIAN_PACKAGE_DEPENDS
-            "libc6 (>=2.11), ffmpeg (>=0.8.9), gstreamer0.10-plugins-base (>=0.10.36), libqtcore4 (>=4.8.1), libqtgui4 (>=4.8.1), libqt4-xml (>=4.8.1), libv4l-0 (>=0.8.6), libgphoto2-2 (>=2.4.11), libgphoto2-port0 (>=2.4.11)"
-        )
-    ENDIF ("${LSB_DISTRIB}" MATCHES "Ubuntu12.04")
-    IF ("${LSB_DISTRIB}" MATCHES "Ubuntu14.04")
-        SET(
-            CPACK_DEBIAN_PACKAGE_DEPENDS
-            "libc6 (>=2.11), libavbin0 (>=7-1), gstreamer0.10-plugins-base (>=0.10.36), libqtcore4 (>=4.8.1), libqtgui4 (>=4.8.1), libqt4-xml (>=4.8.1), libv4l-0 (>=0.8.6), libgphoto2-6 (>=2.5.3)"
-        )
-    ENDIF ("${LSB_DISTRIB}" MATCHES "Ubuntu14.04")
-    IF ("${LSB_DISTRIB}" MATCHES "Debian6.*")
-        SET(
-            CPACK_DEBIAN_PACKAGE_DEPENDS
-            "libc6 (>=2.11), ffmpeg (>=0.8.9), gstreamer0.10-plugins-base (>=0.10.36), libqtcore4 (>=4.8.2), libqtgui4 (>=4.8.2), libqt4-xml (>=4.8.2), libv4l-0 (>=0.8.8), libgphoto2-2 (>=2.4.14), libgphoto2-port0 (>=2.4.14)"
-        )
-    ENDIF ("${LSB_DISTRIB}" MATCHES "Debian6.*")
-    IF ("${LSB_DISTRIB}" MATCHES "Debian7.*")
-        SET(
-            CPACK_DEBIAN_PACKAGE_DEPENDS
-            "libc6 (>=2.11), ffmpeg (>=0.8.9), gstreamer0.10-plugins-base (>=0.10.36), libqtcore4 (>=4.8.2), libqtgui4 (>=4.8.2), libqt4-xml (>=4.8.2), libv4l-0 (>=0.8.8), libgphoto2-2 (>=2.4.14), libgphoto2-port0 (>=2.4.14)"
-        )
-    ENDIF ("${LSB_DISTRIB}" MATCHES "Debian7.*")
-    IF ("${LSB_DISTRIB}" MATCHES "Debian8.*")
-        SET(
-            CPACK_DEBIAN_PACKAGE_DEPENDS
-            "libc6 (>=2.11), libav-tools (>=6:11.3), gstreamer0.10-plugins-base (>=0.10.36), libqtcore4 (>=4.8.2), libqtgui4 (>=4.8.2), libqt4-xml (>=4.8.2), libv4l-0 (>=0.8.8), libgphoto2-6 (>=2.5.3)"
-        )
-    ENDIF ("${LSB_DISTRIB}" MATCHES "Debian8.*")
-    IF (NOT CPACK_DEBIAN_PACKAGE_DEPENDS)
-        MESSAGE ("WARNING: ${LSB_DISTRIB} not supported yet.\nPlease set deps in CMakeLists.txt before packaging.")
-    ENDIF (NOT CPACK_DEBIAN_PACKAGE_DEPENDS)
-ENDIF ("${LSB_DISTRIB}" MATCHES "Ubuntu|Debian")
-MESSAGE("CPACK_DEBIAN_PACKAGE_DEPENDS: ${CPACK_DEBIAN_PACKAGE_DEPENDS}")
-
-# Mandatory
-# CPACK_DEBIAN_PACKAGE_NAME = CPACK_PACKAGE_NAME
-MESSAGE("CPACK_DEBIAN_PACKAGE_NAME: ${CPACK_DEBIAN_PACKAGE_NAME}")
-# CPACK_DEBIAN_PACKAGE_VERSION = CPACK_PACKAGE_VERSION
-MESSAGE("CPACK_DEBIAN_PACKAGE_VERSION: ${CPACK_DEBIAN_PACKAGE_VERSION}")
-# CPACK_DEBIAN_PACKAGE_ARCHITECTURE = dpkg output       # dpkg --print-architecture ???
-MESSAGE("CPACK_DEBIAN_PACKAGE_ARCHITECTURE: ${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
-# CPACK_DEBIAN_PACKAGE_MAINTAINER = CPACK_PACKAGE_CONTACT
-MESSAGE("CPACK_DEBIAN_PACKAGE_MAINTAINER: ${CPACK_DEBIAN_PACKAGE_MAINTAINER}")
-SET(
-    CPACK_DEBIAN_PACKAGE_DESCRIPTION
-    "create stop motion animations\n qStopMotion can be used to create stop motion animations."
-)
-MESSAGE("CPACK_DEBIAN_PACKAGE_DESCRIPTION: ${CPACK_DEBIAN_PACKAGE_DESCRIPTION}")
-SET(
-    CPACK_DEBIAN_PACKAGE_SECTION
-    "graphics"
-)
-MESSAGE("CPACK_DEBIAN_PACKAGE_SECTION: ${CPACK_DEBIAN_PACKAGE_SECTION}")
-SET(
-    CPACK_DEBIAN_PACKAGE_PRIORITY
-    "optional"
-)
-MESSAGE("CPACK_DEBIAN_PACKAGE_PRIORITY: ${CPACK_DEBIAN_PACKAGE_PRIORITY}")
-
-# Options
-
-SET(CPACK_DEBIAN_PACKAGE_HOMEPAGE ${QSTOPMOTION_HOMEPAGE})
-# CPACK_DEBIAN_PACKAGE_SHLIBDEPS
-# SET(CPACK_DEBIAN_PACKAGE_DEBUG "ON")
-# CPACK_DEBIAN_PACKAGE_DEBUG
-# CPACK_DEBIAN_PACKAGE_ENHANCES
-# CPACK_DEBIAN_PACKAGE_BREAKS
-# CPACK_DEBIAN_PACKAGE_CONFLICTS
-# CPACK_DEBIAN_PACKAGE_PROVIDES
-# CPACK_DEBIAN_PACKAGE_REPLACES
-SET(
-    CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
-    "${CMAKE_SOURCE_DIR}/deb/postinst"
-)
-
-###############################################################################
-# RPM package - Linux only (Fedora, Red Hat, CentOS, Mandriva, ...)
-###############################################################################
-
-# Mandatory
-SET(
-    CPACK_RPM_PACKAGE_SUMMARY
-    ${CPACK_PACKAGE_DESCRIPTION_SUMMARY}
-)
-MESSAGE("CPACK_RPM_PACKAGE_SUMMARY: ${CPACK_RPM_PACKAGE_SUMMARY}")
-# CPACK_RPM_PACKAGE_NAME = CPACK_PACKAGE_NAME
-MESSAGE("CPACK_RPM_PACKAGE_NAME: ${CPACK_RPM_PACKAGE_NAME}")
-# CPACK_RPM_PACKAGE_VERSION = CPACK_PACKAGE_VERSION
-MESSAGE("CPACK_RPM_PACKAGE_VERSION: ${CPACK_RPM_PACKAGE_VERSION}")
-# SET(CPACK_RPM_PACKAGE_RELEASE "1")
-MESSAGE("CPACK_RPM_PACKAGE_RELEASE: ${CPACK_RPM_PACKAGE_RELEASE}")
-SET(
-    CPACK_RPM_PACKAGE_LICENSE
-    "GPLv2"
-)
-MESSAGE("CPACK_RPM_PACKAGE_LICENSE: ${CPACK_RPM_PACKAGE_LICENSE}")
-SET(
-    CPACK_RPM_PACKAGE_GROUP
-    "Applications/Multimedia"
-)
-MESSAGE("CPACK_RPM_PACKAGE_GROUP: ${CPACK_RPM_PACKAGE_GROUP}")
-# CPACK_RPM_PACKAGE_VENDOR = CPACK_PACKAGE_VENDOR
-MESSAGE("CPACK_RPM_PACKAGE_VENDOR: ${CPACK_RPM_PACKAGE_VENDOR}")
-# CPACK_RPM_PACKAGE_MAINTAINER = CPACK_PACKAGE_CONTACT
-MESSAGE("CPACK_RPM_PACKAGE_MAINTAINER: ${CPACK_RPM_PACKAGE_MAINTAINER}")
-
-# Options
-# SET(CPACK_RPM_PACKAGE_ARCHITECTURE "noarch")
-SET(
-    CPACK_RPM_PACKAGE_URL
-    ${QSTOPMOTION_HOMEPAGE}
-)
-# CPACK_RPM_PACKAGE_DESCRIPTION = CPACK_PACKAGE_DESCRIPTION_FILE
-# CPACK_RPM_COMPRESSION_TYPE
-# requireses must be in a aphabetic order
-SET(
-    CPACK_RPM_PACKAGE_REQUIRES
-    "glibc >= 2.12, ffmpeg >= 0.8.9, gstreamer >= 0.10.29, qt >= 4.6.2, libv4l >= 0.6.3, libgphoto2 >= 2.4.7"
-)
-MESSAGE("CPACK_RPM_PACKAGE_REQUIRES: ${CPACK_RPM_PACKAGE_REQUIRES}")
-# CPACK_RPM_PACKAGE_SUGGESTS
-# CPACK_RPM_PACKAGE_PROVIDES
-
-# SET(CPACK_RPM_SPEC_INSTALL_POST "desktop-file-validate /usr/share/applications/qstopmotion.desktop")
-
-# SET(CPACK_RPM_PRE_INSTALL_SCRIPT_FILE "pre_install.sh")
-# SET(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "post_install.sh")
-# SET(CPACK_RPM_PRE_UNINSTALL_SCRIPT_FILE "pre_install.sh")
-# SET(CPACK_RPM_POST_UNINSTALL_SCRIPT_FILE "post_install.sh")
-
-# Prevent conflicts with the filesystem package
-SET(
-    CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION
-    "/usr/share/applications"
-    "/usr/share/pixmaps"
-)
-
-###############################################################################
 # Activate packaging functionality
 ###############################################################################
 
