# apply upstream patch to support Ubuntu 15.04
Index: tvbrowser-3.4.1.0/tvbrowser.sh
===================================================================
--- tvbrowser-3.4.1.0.orig/tvbrowser.sh	2014-12-24 10:04:49.000000000 +0100
+++ tvbrowser-3.4.1.0/tvbrowser.sh	2015-04-28 19:25:04.827307377 +0200
@@ -43,6 +43,15 @@
 
 check_version()
 {
+  TOOL_OPTIONS=`echo $JAVA_TOOL_OPTIONS`
+
+  size=${#TOOL_OPTIONS}
+
+  if [[ $size -ne 0 ]] ; then
+    echo "unset JAVA_TOOL_OPTIONS"
+    unset JAVA_TOOL_OPTIONS
+  fi
+
   JAVA_HEADER=`${JAVA_PROGRAM_DIR}java -version 2>&1 | head -n 1`
   JAVA_IMPL=`echo ${JAVA_HEADER} | cut -f1 -d' '`
   if [ "$JAVA_IMPL" = "java" ] ; then
