Set the correct script path.
Index: urd-1.7.0/functions/defaults.php
===================================================================
--- urd-1.7.0.orig/functions/defaults.php	2013-03-02 23:44:17.000000000 +0100
+++ urd-1.7.0/functions/defaults.php	2013-03-04 21:17:54.117356083 +0100
@@ -124,7 +124,7 @@
     if (isset($_SESSION['trickle']))  { $prefArray_root['trickle_path']  = $_SESSION['trickle']; }
     if (isset($_SESSION['subdownloader'])) {  $prefArray_root['subdownloader_path']  = $_SESSION['subdownloader']; }
 
-    $prefArray_root['urdd_path'] = $pathc . 'urdd.sh'; // location where urdd is stored
+    $prefArray_root['urdd_path'] = '/usr/share/urd/urdd.sh'; // location where urdd is stored
     if (isset($_SESSION['urdd'])) {
         $prefArray_root['urdd_path']  = $_SESSION['urdd'];
     }
