Change WORKING_DIR
Index: ledgersmb-1.3.0/ledgersmb-httpd.conf.template
===================================================================
--- ledgersmb-1.3.0.orig/ledgersmb-httpd.conf.template	2011-10-09 13:54:21.917187040 +0200
+++ ledgersmb-1.3.0/ledgersmb-httpd.conf.template	2011-10-09 13:56:06.937187076 +0200
@@ -1,8 +1,11 @@
 # This file must be edited to substitute the current path with WORKING_DIR
+Alias /ledgersmb/doc/LedgerSMB-manual.pdf /usr/share/doc/ledgersmb/LedgerSMB-manual.pdf
+<Files /usr/share/doc/ledgersmb/LedgerSMB-manual.pdf>
+</Files>
 
-Alias /ledgersmb WORKING_DIR/
+Alias /ledgersmb /usr/share/ledgersmb
 
-<Directory WORKING_DIR>
+<Directory /usr/share/ledgersmb>
 
   # Rewrite rule to allow HTTP Authorization information to the scripts only
   # from this directory.
@@ -36,32 +39,32 @@
   </Files>
 </Directory>
 
-<Directory WORKING_DIR/users>
+<Directory /usr/share/ledgersmb/users>
   Order Deny,Allow
   Deny from All
 </Directory>
 
-<Directory WORKING_DIR/bin>
+<Directory /usr/share/ledgersmb/bin>
   Order Deny,Allow
   Deny from All
 </Directory>
 
-<Directory WORKING_DIR/utils>
+<Directory /usr/share/ledgersmb/utils>
   Order Deny,Allow
   Deny from All
 </Directory>
 
-<Directory WORKING_DIR/spool>
+<Directory /usr/share/ledgersmb/spool>
   Order Deny,Allow
   Deny from All
 </Directory>
 
-<Directory WORKING_DIR/templates>
+<Directory /usr/share/ledgersmb/templates>
   Order Deny,Allow
   Deny from All
 </Directory>
 
-<Directory WORKING_DIR/LedgerSMB>
+<Directory /usr/share/ledgersmb/LedgerSMB>
   Order Deny,Allow
   Deny from All
 </Directory>
