Set the correct path to the contrib directory.
Index: ledgersmb-1.3.15/ledgersmb.conf.default
===================================================================
--- ledgersmb-1.3.15.orig/ledgersmb.conf.default	2012-03-28 07:41:32.000000000 +0200
+++ ledgersmb-1.3.15/ledgersmb.conf.default	2012-04-17 19:00:00.694008577 +0200
@@ -83,7 +83,7 @@
 db_namespace = public
 #contrib_dir PG_CONTRIB_DIR 
 #with postgresql-9.1 this stuff is in /usr/share/postgresql/9.1/extension and is injected in database with create extension pg_trgm,btree_gist...
-contrib_dir = /usr/share/pgsql/contrib/
+contrib_dir = /usr/share/postgresql/9.1/extension/
 # sslmode can be require, allow, prefer, or disable.  Defaults to prefer.
 sslmode = prefer
 
