Index: avidemux-2.5.2/cmake/admDetermineSystem.cmake
===================================================================
--- avidemux-2.5.2.orig/cmake/admDetermineSystem.cmake	2009-12-27 17:21:35.000000000 -0500
+++ avidemux-2.5.2/cmake/admDetermineSystem.cmake	2009-12-27 17:22:42.000000000 -0500
@@ -119,9 +119,9 @@
 
 			PERFORM_SYSTEM_TEST(cpu_altivec_check.cpp "AltiVec capable" ALTIVEC_SUPPORTED "${ADM_ALTIVEC_FLAGS}")
 
-			IF (ALTIVEC_SUPPORTED)
-				SET(ADM_CPU_ALTIVEC 1)
-			ENDIF (ALTIVEC_SUPPORTED)
+			#IF (ALTIVEC_SUPPORTED)
+			#	SET(ADM_CPU_ALTIVEC 1)
+			#ENDIF (ALTIVEC_SUPPORTED)
 
 			PERFORM_SYSTEM_TEST(cpu_dcbzl_check.cpp "dcbzl capable" DCBZL_SUPPORTED)
 
