diff -Nur avidemux-2.5.3.orig/cmake/admCheckXvid.cmake avidemux-2.5.3/cmake/admCheckXvid.cmake
--- avidemux-2.5.3.orig/cmake/admCheckXvid.cmake	2008-06-13 19:18:36.000000000 +0700
+++ avidemux-2.5.3/cmake/admCheckXvid.cmake	2010-08-08 08:21:07.932283998 +0700
@@ -6,7 +6,7 @@
 		MESSAGE(STATUS "*****************")
 
 		IF (XVID)
-			FIND_HEADER_AND_LIB(XVID xvid.h xvidcore xvid_plugin_single)
+			FIND_HEADER_AND_LIB(XVID xvid.h xvidcore)
 			PRINT_LIBRARY_INFO("Xvid" XVID_FOUND "${XVID_INCLUDE_DIR}" "${XVID_LIBRARY_DIR}")
 		ELSE (XVID)
 			MESSAGE("${MSG_DISABLE_OPTION}")
