Set the correct paths.
Index: wakeonplan-0.0.0+bzr61/wakeonplan.desktop.in
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan.desktop.in	2012-09-28 21:40:10.662635497 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan.desktop.in	2012-09-28 21:41:29.690633145 +0200
@@ -2,8 +2,8 @@
 _Name=Wakeonplan
 _Comment=Wakeonplan application
 Categories=GNOME;Utility;
-Exec=/opt/extras.ubuntu.com/wakeonplan/bin/wakeonplan
-Icon=/usr/share/wakeonplan/media/wakeonplan.svg
+Exec=wakeonplan
+Icon=wakeonplan
 Terminal=false
 Type=Application
 
@@ -11,16 +11,16 @@
 
 [Desktop Action Set_default]
 Name=Set ACPI alarm to the default time
-Exec=/opt/extras.ubuntu.com/wakeonplan/bin/wakeonplan -s
+Exec=wakeonplan -s
 
 [Desktop Action Set_relative]
 Name=Set ACPI alarm in relative interval
-Exec=/opt/extras.ubuntu.com/wakeonplan/bin/wakeonplan -a
+Exec=wakeonplan -a
 
 [Desktop Action Unset]
 Name=Unset the ACPI alarm
-Exec=/opt/extras.ubuntu.com/wakeonplan/bin/wakeonplan -u
+Exec=wakeonplan -u
 
 [Desktop Action Show]
 Name=Show current alarm state
-Exec=/opt/extras.ubuntu.com/wakeonplan/bin/wakeonplan -r
\ No newline at end of file
+Exec=wakeonplan -r
