# just fix a typo
Index: patchelfmod-0.14/man/patchelfmod.1.in
===================================================================
--- patchelfmod-0.14.orig/man/patchelfmod.1.in	2014-06-13 16:28:42.000000000 +0200
+++ patchelfmod-0.14/man/patchelfmod.1.in	2014-06-22 00:36:15.037139655 +0200
@@ -72,11 +72,11 @@
 .TP
 \fB\-a\fR, \fB\-\-add\-needed\fR <library>[,<library>...]
 Add a dependency <library> on a dynamic library.
-Can be a single library or a comma seperated list.
+Can be a single library or a comma separated list.
 .TP
 \fB\-r\fR, \fB\-\-remove\-needed\fR <library>[,<library>...]
 Remove a dependency <library> from a dynamic library.
-Can be a single library or a comma seperated list.
+Can be a single library or a comma separated list.
 .TP
 \fB\-n\fR, \fB\-\-replace\-needed\fR <library>,<new\-library>
 Replace a dependency <library> on a dynamic library with
