# do not search Qt includes locally.
Index: clipgrab-3.6.1/video_youtube.cpp
===================================================================
--- clipgrab-3.6.1.orig/video_youtube.cpp	2016-06-27 18:21:49.888315299 +0200
+++ clipgrab-3.6.1/video_youtube.cpp	2016-06-27 18:21:49.888315299 +0200
@@ -22,7 +22,7 @@
 
 
 #include "video_youtube.h"
-#include "QMutableListIterator"
+#include <QMutableListIterator>
 
 video_youtube::video_youtube()
 {
Index: clipgrab-3.6.1/video_youtube.h
===================================================================
--- clipgrab-3.6.1.orig/video_youtube.h	2016-06-27 18:21:49.888315299 +0200
+++ clipgrab-3.6.1/video_youtube.h	2016-06-27 18:22:29.297186750 +0200
@@ -25,12 +25,12 @@
 #define video_YOUTUBE_H
 
 #include "video.h"
-#include "QtWebKit"
+#include <QtWebKit>
 #include <QtGui>
 #include "QDomDocument"
 #include "ui_login_dialog.h"
 
-#include "QFile"
+#include <QFile>
 
 struct fmtQuality
 {
