From df6d54429261cb9916fbf83aa33d41144120380c Mon Sep 17 00:00:00 2001
From: Christoph Korn <korn@ubuntu.(none)>
Date: Mon, 14 Dec 2009 22:11:13 +0100
Subject: [PATCH] Makefile

---
 fotoxx-9.0/Makefile |    7 +++----
 1 files changed, 3 insertions(+), 4 deletions(-)

Index: fotoxx-14.05/Makefile
===================================================================
--- fotoxx-14.05.orig/Makefile	2014-04-30 23:38:01.031346128 +0200
+++ fotoxx-14.05/Makefile	2014-04-30 23:38:39.715345797 +0200
@@ -14,7 +14,7 @@
 DATADIR = $(SHAREDIR)/data
 ICONDIR = $(SHAREDIR)/icons
 LOCALESDIR = $(SHAREDIR)/locales
-DOCDIR = $(PREFIX)/share/doc/fotoxx
+DOCDIR = $(PREFIX)/share/doc/fotoxx/extras
 MANDIR = $(PREFIX)/share/man/man1
 MENUFILE = $(PREFIX)/share/applications/fotoxx.desktop
 
@@ -77,7 +77,6 @@
 	rm -f -R  $(DESTDIR)$(SHAREDIR)
 	rm -f -R  $(DESTDIR)$(DOCDIR)
 	rm -f  $(DESTDIR)$(MANDIR)/fotoxx.1.gz
-	xdg-desktop-menu uninstall $(DESTDIR)$(MENUFILE)
 	rm -f  $(DESTDIR)$(MENUFILE)
 	mkdir -p  $(DESTDIR)$(BINDIR)
 	mkdir -p  $(DESTDIR)$(DATADIR)
@@ -100,14 +99,12 @@
 	# menu (desktop) file
 	cp -f desktop $(DESTDIR)$(MENUFILE)
 	chmod +x $(DESTDIR)$(MENUFILE)
-	xdg-desktop-menu install --novendor $(DESTDIR)$(MENUFILE)
 
 uninstall:
 	rm -f  $(DESTDIR)$(BINDIR)/fotoxx
 	rm -f -R  $(DESTDIR)$(SHAREDIR)
 	rm -f -R  $(DESTDIR)$(DOCDIR)
 	rm -f  $(DESTDIR)$(MANDIR)/fotoxx.1.gz
-	xdg-desktop-menu uninstall $(DESTDIR)$(MENUFILE)
 	rm -f  $(DESTDIR)$(MENUFILE)
 
 clean: 
