From df6d54429261cb9916fbf83aa33d41144120380c Mon Sep 17 00:00:00 2001
From: Christoph Korn <korn@ubuntu.(none)>
Date: Mon, 14 Dec 2009 22:11:13 +0100
Subject: [PATCH] Makefile

---
 fotoxx-9.0/Makefile |    7 +++----
 1 files changed, 3 insertions(+), 4 deletions(-)

Index: fotoxx-17.01.1/Makefile
===================================================================
--- fotoxx-17.01.1.orig/Makefile	2017-01-12 18:03:13.502370246 +0100
+++ fotoxx-17.01.1/Makefile	2017-01-12 18:03:13.498370282 +0100
@@ -13,7 +13,7 @@
 DATADIR = $(SHAREDIR)/data
 ICONDIR = $(SHAREDIR)/icons
 LOCALESDIR = $(SHAREDIR)/locales
-DOCDIR = $(PREFIX)/share/doc/fotoxx
+DOCDIR = $(PREFIX)/share/doc/fotoxx/extras
 MANDIR = $(PREFIX)/share/man/man1
 APPDATADIR = $(PREFIX)/share/appdata
 MENUFILE = $(PREFIX)/share/applications/fotoxx.desktop
