# fix icon path
Index: fotoxx-17.08.1/fotoxx.desktop
===================================================================
--- fotoxx-17.08.1.orig/fotoxx.desktop	2017-08-06 14:34:59.451540021 +0200
+++ fotoxx-17.08.1/fotoxx.desktop	2017-08-06 14:34:59.443540152 +0200
@@ -8,7 +8,7 @@
 Terminal=false
 MimeType=image/bmp;image/gif;image/tiff;image/jpeg;image/png;
 Exec=fotoxx %f
-Icon=/usr/share/icons/fotoxx.png
+Icon=/usr/share/fotoxx/icons/fotoxx.png
 
 GenericName[de]=Foto Bearbeitung
 GenericName[fr]=Gestionnaire de photos
