We put the documentation in an own directory.
Index: fotoxx-16.05/zfuncs.cc
===================================================================
--- fotoxx-16.05.orig/zfuncs.cc	2016-04-19 22:04:16.438584905 +0200
+++ fotoxx-16.05/zfuncs.cc	2016-04-19 22:04:16.434584812 +0200
@@ -4315,7 +4315,7 @@
    strncatv(zdatadir,199,work,"/share/",zappname,"/data",null);                  //  /prefix/share/appname/data
    strncatv(zicondir,199,work,"/share/",zappname,"/icons",null);                 //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,work,"/share/",zappname,"/locales",null);            //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,work,"/share/doc/",zappname,null);                       //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,work,"/share/doc/",zappname,"/extras/",null);                       //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                              //  flexible DOCDIR location
