We put the documentation in an own directory.
Index: fotoxx-16.10/zfuncs.cc
===================================================================
--- fotoxx-16.10.orig/zfuncs.cc	2016-10-01 15:47:04.412207417 +0200
+++ fotoxx-16.10/zfuncs.cc	2016-10-01 15:47:04.408207333 +0200
@@ -4370,7 +4370,7 @@
    strncatv(zdatadir,199,zprefix,"/share/",zappname,"/data",null);               //  /prefix/share/appname/data
    strncatv(zicondir,199,zprefix,"/share/",zappname,"/icons",null);              //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,zprefix,"/share/",zappname,"/locales",null);         //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,null);                    //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,"/extras/",null);                    //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                              //  flexible DOCDIR location (SUSE)    6.5
