We put the documentation in an own directory.
Index: fotoxx-17.01/zfuncs.cc
===================================================================
--- fotoxx-17.01.orig/zfuncs.cc	2017-01-01 13:49:11.571605125 +0100
+++ fotoxx-17.01/zfuncs.cc	2017-01-01 13:49:11.571605125 +0100
@@ -4420,7 +4420,7 @@
    strncatv(zdatadir,199,zprefix,"/share/",zappname,"/data",null);               //  /prefix/share/appname/data
    strncatv(zicondir,199,zprefix,"/share/",zappname,"/icons",null);              //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,zprefix,"/share/",zappname,"/locales",null);         //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,null);                    //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,"/extras/",null);                    //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                              //  flexible DOCDIR location (SUSE)    6.5
