We put the documentation in an own directory.
Index: fotoxx-15.04/zfuncs.cc
===================================================================
--- fotoxx-15.04.orig/zfuncs.cc	2015-04-01 19:42:47.459325779 +0200
+++ fotoxx-15.04/zfuncs.cc	2015-04-01 19:44:47.747322201 +0200
@@ -4263,7 +4263,7 @@
    strncatv(zdatadir,199,work,"/share/",zappname,"/data",null);            //  /prefix/share/appname/data
    strncatv(zicondir,199,work,"/share/",zappname,"/icons",null);           //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,work,"/share/",zappname,"/locales",null);      //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,work,"/share/doc/",zappname,null);                 //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,work,"/share/doc/",zappname,"/extras/",null);                 //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                        //  flexible DOCDIR location
