We put the documentation in an own directory.
Index: fotoxx-16.11/zfuncs.cc
===================================================================
--- fotoxx-16.11.orig/zfuncs.cc	2016-11-06 03:48:43.475578830 +0100
+++ fotoxx-16.11/zfuncs.cc	2016-11-06 03:48:43.471578746 +0100
@@ -4384,7 +4384,7 @@
    strncatv(zdatadir,199,zprefix,"/share/",zappname,"/data",null);               //  /prefix/share/appname/data
    strncatv(zicondir,199,zprefix,"/share/",zappname,"/icons",null);              //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,zprefix,"/share/",zappname,"/locales",null);         //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,null);                    //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,zprefix,"/share/doc/",zappname,"/extras/",null);                    //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                              //  flexible DOCDIR location (SUSE)    6.5
