We put the documentation in an own directory.
Index: fotoxx-13.03.1/zfuncs.cc
===================================================================
--- fotoxx-13.03.1.orig/zfuncs.cc	2013-03-09 14:48:01.000000000 +0100
+++ fotoxx-13.03.1/zfuncs.cc	2013-03-09 17:10:58.937159158 +0100
@@ -3914,7 +3914,7 @@
    strncatv(zdatadir,199,work,"/share/",zappname,"/data",null);            //  /prefix/share/appname/data
    strncatv(zicondir,199,work,"/share/",zappname,"/icons",null);           //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,work,"/share/",zappname,"/locales",null);      //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,work,"/share/doc/",zappname,null);                 //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,work,"/share/doc/",zappname,"/extras/",null);                 //  /prefix/share/doc/appname/extras
    
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                        //  flexible DOCDIR location
