We put the documentation in an own directory.
Index: fotoxx-16.02.1/zfuncs.cc
===================================================================
--- fotoxx-16.02.1.orig/zfuncs.cc	2016-02-05 18:54:01.059258937 +0100
+++ fotoxx-16.02.1/zfuncs.cc	2016-02-05 18:54:01.055258828 +0100
@@ -4327,7 +4327,7 @@
    strncatv(zdatadir,199,work,"/share/",zappname,"/data",null);                  //  /prefix/share/appname/data
    strncatv(zicondir,199,work,"/share/",zappname,"/icons",null);                 //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,work,"/share/",zappname,"/locales",null);            //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,work,"/share/doc/",zappname,null);                       //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,work,"/share/doc/",zappname,"/extras/",null);                       //  /prefix/share/doc/appname
 
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                              //  flexible DOCDIR location
