We put the documentation in an own directory.
Index: fotoxx-15.03/zfuncs.cc
===================================================================
--- fotoxx-15.03.orig/zfuncs.cc	2015-02-28 17:19:31.919654117 +0100
+++ fotoxx-15.03/zfuncs.cc	2015-02-28 17:19:31.919654117 +0100
@@ -4265,7 +4265,7 @@
    strncatv(zdatadir,199,work,"/share/",zappname,"/data",null);            //  /prefix/share/appname/data
    strncatv(zicondir,199,work,"/share/",zappname,"/icons",null);           //  /prefix/share/appname/icons
    strncatv(zlocalesdir,199,work,"/share/",zappname,"/locales",null);      //  /prefix/share/appname/locales
-   strncatv(zdocdir,199,work,"/share/doc/",zappname,null);                 //  /prefix/share/doc/appname
+   strncatv(zdocdir,199,work,"/share/doc/",zappname,"/extras/",null);                 //  /prefix/share/doc/appname/extras
    
    #ifdef DOCDIR
       strncpy0(zdocdir,DOCDIR,199);                                        //  flexible DOCDIR location
