Description: Update list of Ubuntu releases
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Forwarded: not yet
Updated: 2014-06-20

--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -166,6 +166,9 @@
         "ubuntuquantal" : "ubuntu12.10",
         "ubunturaring" : "ubuntu13.04",
         "ubuntusaucy" : "ubuntu13.10",
+        "ubuntutrusty" : "ubuntu14.04",
+        "ubuntuutopic" : "ubuntu14.10",
+        "ubuntuvivid" : "ubuntu15.04",
         "virtio26": "fedora10",
         "vista" : "winvista",
         "winxp64" : "winxp",
