From: Oliver Seufer <debian@seufer.de>
Date: Sat, 1 Jun 2013 22:41:07 +0200
Subject: fix path to hvmloader

Closes: #685749
---
 virtinst/guest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: virt-manager-1.4.2/virtinst/guest.py
===================================================================
--- virt-manager-1.4.2.orig/virtinst/guest.py	2017-08-09 17:50:01.910602495 +0200
+++ virt-manager-1.4.2/virtinst/guest.py	2017-08-09 17:50:55.045846872 +0200
@@ -812,7 +812,7 @@
             self.os.init = self.os.init or "/bin/sh"
 
         if not self.os.loader and self.os.is_hvm() and self.type == "xen":
-            self.os.loader = "/usr/lib/xen/boot/hvmloader"
+            self.os.loader = "hvmloader"
         if self.os.kernel or self.os.init:
             self.os.bootorder = []
 
