From: Stephane Glondu <steph@glondu.net>
Date: Sat, 23 Jul 2011 17:22:12 +0200
Subject: Tune grisbi.desktop

---
 share/grisbi.desktop.in |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/share/grisbi.desktop.in b/share/grisbi.desktop.in
index 2941a13..b74a394 100644
--- a/share/grisbi.desktop.in
+++ b/share/grisbi.desktop.in
@@ -2,7 +2,9 @@
 _Name=Grisbi
 _Comment=Personnal finances manager
 Exec=grisbi
-Icon=grisbi
+Icon=/usr/share/pixmaps/grisbi/grisbi.xpm
 Terminal=false
 Type=Application
-Categories=Office;Finance;
+Categories=Application;Office;Finance;
+MimeType=application/x-grisbi
+StartupNotify=true
-- 
