Fix some spelling mistakes.
Index: gogglesmm-0.12.6/ChangeLog
===================================================================
--- gogglesmm-0.12.6.orig/ChangeLog	2012-01-05 03:09:48.000000000 +0100
+++ gogglesmm-0.12.6/ChangeLog	2012-01-28 12:33:45.306814136 +0100
@@ -51,7 +51,7 @@
   - Properly initialize libgcrypt to avoid warnings.
   - Added button to get out of miniplayer mode and make the miniplayer behave the same as the regular browser window. (issue 196).
   - New additional shortcut to toggle between miniplayer mode and browser mode (Ctrl-M).
-  - Miniplayer is resizeable and remember its size and position. It will no longer automatically resize itself.
+  - Miniplayer is resizable and remember its size and position. It will no longer automatically resize itself.
   - Fix crash when importing with specific icon themes selected. (issue 218)
   - Add big-endian support to flac album art loading. (issue 218)
 
