# tests require a display
Index: corebird-1.2/Makefile.am
===================================================================
--- corebird-1.2.orig/Makefile.am	2016-02-29 21:36:28.500934620 +0100
+++ corebird-1.2/Makefile.am	2016-02-29 21:37:00.669604259 +0100
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-SUBDIRS = src ui data assets po tests
+SUBDIRS = src ui data assets po
 
 EXTRA_DIST = \
 	vapi/config.vapi \
