# tests require a display
Index: corebird-1.3/Makefile.am
===================================================================
--- corebird-1.3.orig/Makefile.am	2016-08-05 21:54:21.582075893 +0200
+++ corebird-1.3/Makefile.am	2016-08-05 21:54:30.434295489 +0200
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-SUBDIRS = src data po tests
+SUBDIRS = src data po
 
 EXTRA_DIST = \
 	vapi/config.vapi \
