---
 configure |   95 --------------------------------------------------------------
 1 file changed, 95 deletions(-)

Index: ogmrip-1.0.1/configure.in
===================================================================
--- ogmrip-1.0.1.orig/configure.in	2015-01-31 22:13:08.108606817 +0100
+++ ogmrip-1.0.1/configure.in	2015-01-31 22:13:08.096606818 +0100
@@ -246,17 +246,17 @@
 
 dnl **************************************************************
 
-AC_PATH_PROG(MPLAYER_PROG, mplayer)
-if test x"$MPLAYER_PROG" = x; then
-  AC_MSG_ERROR(Unable to find mplayer in the PATH. You need mplayer to use OGMRip. Find it on http://www.mplayerhq.hu)
-fi
+dnl AC_PATH_PROG(MPLAYER_PROG, mplayer)
+dnl if test x"$MPLAYER_PROG" = x; then
+dnl   AC_MSG_ERROR(Unable to find mplayer in the PATH. You need mplayer to use OGMRip. Find it on http://www.mplayerhq.hu)
+dnl fi
 
 dnl **************************************************************
 
-AC_PATH_PROG(MENCODER_PROG, mencoder)
-if test x"$MENCODER_PROG" = x; then
-  AC_MSG_ERROR(Unable to find mencoder in the PATH. You need mencoder to use OGMRip. Find it on http://www.mplayerhq.hu)
-fi
+dnl AC_PATH_PROG(MENCODER_PROG, mencoder)
+dnl if test x"$MENCODER_PROG" = x; then
+dnl   AC_MSG_ERROR(Unable to find mencoder in the PATH. You need mencoder to use OGMRip. Find it on http://www.mplayerhq.hu)
+dnl fi
 
 dnl **************************************************************
 
Index: ogmrip-1.0.1/configure
===================================================================
--- ogmrip-1.0.1.orig/configure	2015-01-31 22:13:08.108606817 +0100
+++ ogmrip-1.0.1/configure	2015-01-31 22:13:08.104606818 +0100
@@ -13887,96 +13887,6 @@
 fi
 
 
-# Extract the first word of "mplayer", so it can be a program name with args.
-set dummy mplayer; ac_word=$2
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for $ac_word" >&5
-$as_echo_n "checking for $ac_word... " >&6; }
-if ${ac_cv_path_MPLAYER_PROG+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  case $MPLAYER_PROG in
-  [\\/]* | ?:[\\/]*)
-  ac_cv_path_MPLAYER_PROG="$MPLAYER_PROG" # Let the user override the test with a path.
-  ;;
-  *)
-  as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in $PATH
-do
-  IFS=$as_save_IFS
-  test -z "$as_dir" && as_dir=.
-    for ac_exec_ext in '' $ac_executable_extensions; do
-  if as_fn_executable_p "$as_dir/$ac_word$ac_exec_ext"; then
-    ac_cv_path_MPLAYER_PROG="$as_dir/$ac_word$ac_exec_ext"
-    $as_echo "$as_me:${as_lineno-$LINENO}: found $as_dir/$ac_word$ac_exec_ext" >&5
-    break 2
-  fi
-done
-  done
-IFS=$as_save_IFS
-
-  ;;
-esac
-fi
-MPLAYER_PROG=$ac_cv_path_MPLAYER_PROG
-if test -n "$MPLAYER_PROG"; then
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: $MPLAYER_PROG" >&5
-$as_echo "$MPLAYER_PROG" >&6; }
-else
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-fi
-
-
-if test x"$MPLAYER_PROG" = x; then
-  as_fn_error $? "Unable to find mplayer in the PATH. You need mplayer to use OGMRip. Find it on http://www.mplayerhq.hu" "$LINENO" 5
-fi
-
-
-# Extract the first word of "mencoder", so it can be a program name with args.
-set dummy mencoder; ac_word=$2
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for $ac_word" >&5
-$as_echo_n "checking for $ac_word... " >&6; }
-if ${ac_cv_path_MENCODER_PROG+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  case $MENCODER_PROG in
-  [\\/]* | ?:[\\/]*)
-  ac_cv_path_MENCODER_PROG="$MENCODER_PROG" # Let the user override the test with a path.
-  ;;
-  *)
-  as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in $PATH
-do
-  IFS=$as_save_IFS
-  test -z "$as_dir" && as_dir=.
-    for ac_exec_ext in '' $ac_executable_extensions; do
-  if as_fn_executable_p "$as_dir/$ac_word$ac_exec_ext"; then
-    ac_cv_path_MENCODER_PROG="$as_dir/$ac_word$ac_exec_ext"
-    $as_echo "$as_me:${as_lineno-$LINENO}: found $as_dir/$ac_word$ac_exec_ext" >&5
-    break 2
-  fi
-done
-  done
-IFS=$as_save_IFS
-
-  ;;
-esac
-fi
-MENCODER_PROG=$ac_cv_path_MENCODER_PROG
-if test -n "$MENCODER_PROG"; then
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: $MENCODER_PROG" >&5
-$as_echo "$MENCODER_PROG" >&6; }
-else
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-fi
-
-
-if test x"$MENCODER_PROG" = x; then
-  as_fn_error $? "Unable to find mencoder in the PATH. You need mencoder to use OGMRip. Find it on http://www.mplayerhq.hu" "$LINENO" 5
-fi
-
-
 # Check whether --enable-ogm-support was given.
 if test "${enable_ogm_support+set}" = set; then :
   enableval=$enable_ogm_support; have_ogm_support="$enableval"
