# add format string
Index: opentooz-1.1.1/toonz/sources/common/tsystem/tsystempd.cpp
===================================================================
--- opentooz-1.1.1.orig/toonz/sources/common/tsystem/tsystempd.cpp	2016-08-08 08:44:05.000000000 +0200
+++ opentooz-1.1.1/toonz/sources/common/tsystem/tsystempd.cpp	2016-08-09 23:40:52.438130771 +0200
@@ -136,7 +136,7 @@
   cerr << s << endl;
 #endif
 #else
-  qDebug(s.c_str());
+  qDebug("%s", s.c_str());
 #endif
 }
 
Index: opentooz-1.1.1/toonz/sources/toonzlib/tlog.cpp
===================================================================
--- opentooz-1.1.1.orig/toonz/sources/toonzlib/tlog.cpp	2016-08-08 08:44:05.000000000 +0200
+++ opentooz-1.1.1/toonz/sources/toonzlib/tlog.cpp	2016-08-10 00:10:38.869536049 +0200
@@ -92,7 +92,7 @@
   DeregisterEventSource(handle);
 
 #else
-  syslog(Level2XPriority(level), msg.c_str());
+  syslog(Level2XPriority(level), "%s", msg.c_str());
 #endif
 }
 
Index: opentooz-1.1.1/toonz/sources/toonzfarm/tfarm/tlog.cpp
===================================================================
--- opentooz-1.1.1.orig/toonz/sources/toonzfarm/tfarm/tlog.cpp	2016-08-08 08:44:05.000000000 +0200
+++ opentooz-1.1.1/toonz/sources/toonzfarm/tfarm/tlog.cpp	2016-08-10 00:20:31.272152578 +0200
@@ -91,7 +91,7 @@
 
 #else
   std::string str = msg.toStdString();
-  syslog(Level2XPriority(level), str.c_str());
+  syslog(Level2XPriority(level), "%s", str.c_str());
 #endif
 }
 
