# https://github.com/opentoonz/opentoonz/pull/899
Index: opentoonz-1.1.1/toonz/cmake/FindLZO.cmake
===================================================================
--- opentoonz-1.1.1.orig/toonz/cmake/FindLZO.cmake	2016-08-08 08:44:05.000000000 +0200
+++ opentoonz-1.1.1/toonz/cmake/FindLZO.cmake	2016-11-06 03:00:52.983723840 +0100
@@ -9,12 +9,14 @@
         lzo/2.03/include/lzo
 )
 
+# List dynamic library 'so' first
+# since static has issues on some systems, see: #866
 find_library(
     LZO_LIBRARY
     NAMES
+        liblzo2.so
         liblzo2.a
         lzo2_64.lib
-        liblzo2.so
     HINTS
         ${THIRDPARTY_LIBS_HINTS}
     PATH_SUFFIXES
