UbuntuOne should be the default sync option.
Index: tomboy/Tomboy/Addins/WebSyncService/WebSyncServiceAddin.cs
===================================================================
--- tomboy.orig/Tomboy/Addins/WebSyncService/WebSyncServiceAddin.cs	2011-09-26 22:25:08.135886195 +0100
+++ tomboy/Tomboy/Addins/WebSyncService/WebSyncServiceAddin.cs	2011-09-26 23:37:26.202367732 +0100
@@ -97,6 +97,8 @@
 			string serverPref;
 			Api.OAuth oauth;
 			GetConfigSettings (out oauth, out serverPref);
+			if (string.IsNullOrEmpty (serverPref))
+				serverPref = "https://one.ubuntu.com/notes/";
 			prefsWidget = new WebSyncPreferencesWidget (oauth, serverPref, requiredPrefChanged);
 			return prefsWidget;
 		}
