Set UbuntuOne note as start content.
Index: tomboy-1.11.1/Tomboy/NoteManager.cs
===================================================================
--- tomboy-1.11.1.orig/Tomboy/NoteManager.cs	2012-05-02 18:57:38.917271568 +0200
+++ tomboy-1.11.1/Tomboy/NoteManager.cs	2012-05-02 18:57:47.181271322 +0200
@@ -298,6 +298,24 @@
 			                           "Also, if you type the name of another note in your " +
 			                           "current note, it will automatically be linked for you." +
 			                           "</note-content>");
+			string u1_note_content =
+				Catalog.GetString ("<note-content>" +
+                                                  "Ubuntu One\n\n" +
+						   "<bold>Tomboy is better with Ubuntu One</bold> - " +
+						   "the personal cloud that brings your digital life together.\n\n" +
+						   "Ubuntu One does more than sync your files - whether you need " +
+						   "to access your contacts, notes or bookmarks from any computer or " +
+						   "the web, enjoy your favorite music from a cloud integrated store " +
+						   "or stream your entire collection to iPhone and Android mobile phones " +
+						   "- we've raised the bar on personal clouds. Learn more at " +
+						   "<link:url>http://one.ubuntu.com/</link:url>.\n\n" +
+						   "Already have an Ubuntu One account? Setup Tomboy to sync with your " +
+						   "personal cloud by clicking Edit > Preferences and then the Synchronization "+
+						   "tab. Select \"Tomboy Web\" from the list of services and then click the "+
+						   "\"Connect to Server\" button. A web browser will appear with a few " +
+						   "additional instructions.\n\n" +
+						   "You can even enable automatic sync with Ubuntu One!" +
+						   "</note-content>");
 
 			try {
 				Note start_note = Create (Catalog.GetString ("Start Here"),
@@ -309,6 +327,9 @@
 				                          links_note_content);
 				links_note.QueueSave (ChangeType.ContentChanged);
 
+				Note u1_note = Create ("Ubuntu One", u1_note_content);
+				u1_note.QueueSave (ChangeType.ContentChanged);
+
 				if (!Tomboy.IsPanelApplet)
 					start_note.Window.Show ();
 			} catch (Exception e) {
