Index: fritzing-0.8.4b/phoenix.pro
===================================================================
--- fritzing-0.8.4b.orig/phoenix.pro	2013-12-15 19:44:35.682498807 +0100
+++ fritzing-0.8.4b/phoenix.pro	2013-12-15 19:44:35.678498808 +0100
@@ -53,6 +53,12 @@
         LIBS += /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
 }
 unix {
+    # hardening flags
+    QMAKE_CPPFLAGS *=$(shell dpkg-buildflags --get CPPFLAGS)
+    QMAKE_CFLAGS *= $(shell dpkg-buildflags --get CFLAGS)
+    QMAKE_CXXFLAGS *=$(shell dpkg-buildflags --get CXXFLAGS)
+    QMAKE_LDFLAGS *=$(shell dpkg-buildflags --get LDFLAGS)
+
     !macx { # unix is defined on mac
         HARDWARE_PLATFORM = $$system(uname -m)
         contains( HARDWARE_PLATFORM, x86_64 ) {
