Description: Hardening patch
Index: fritzing-0.9.0b/phoenix.pro
===================================================================
--- fritzing-0.9.0b.orig/phoenix.pro	2014-09-29 21:29:04.317242016 +0200
+++ fritzing-0.9.0b/phoenix.pro	2014-09-29 21:29:04.313242016 +0200
@@ -103,6 +103,13 @@
         LIBS += /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
 }
 unix {
+    # hardening flags
+    QMAKE_CPPFLAGS *=$(shell dpkg-buildflags --get CPPFLAGS)
+    QMAKE_CFLAGS *= $(shell dpkg-buildflags --get CFLAGS)
+    QMAKE_CXXFLAGS *=$(shell dpkg-buildflags --get CXXFLAGS)
+    QMAKE_LDFLAGS *=$(shell dpkg-buildflags --get LDFLAGS)
+
+
     !macx { # unix is defined on mac
         HARDWARE_PLATFORM = $$system(uname -m)
         contains( HARDWARE_PLATFORM, x86_64 ) {
