Index: fritzing-0.8.6b/phoenix.pro
===================================================================
--- fritzing-0.8.6b.orig/phoenix.pro	2014-01-22 19:58:56.510025522 +0100
+++ fritzing-0.8.6b/phoenix.pro	2014-01-22 19:58:56.506025522 +0100
@@ -59,6 +59,12 @@
         LIBS += /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
 }
 unix {
+    # hardening flags
+    QMAKE_CPPFLAGS *=$(shell dpkg-buildflags --get CPPFLAGS)
+    QMAKE_CFLAGS *= $(shell dpkg-buildflags --get CFLAGS)
+    QMAKE_CXXFLAGS *=$(shell dpkg-buildflags --get CXXFLAGS)
+    QMAKE_LDFLAGS *=$(shell dpkg-buildflags --get LDFLAGS)
+
     !macx { # unix is defined on mac
         HARDWARE_PLATFORM = $$system(uname -m)
         contains( HARDWARE_PLATFORM, x86_64 ) {
