Index: fritzing-0.8.3b/src/partseditor/pemainwindow.cpp
===================================================================
--- fritzing-0.8.3b.orig/src/partseditor/pemainwindow.cpp	2013-08-15 20:23:37.430572568 +0200
+++ fritzing-0.8.3b/src/partseditor/pemainwindow.cpp	2013-08-15 20:23:37.426572568 +0200
@@ -2402,10 +2402,10 @@
 
     QPointF p = pegi->terminalPoint();
     if (coord == "x") {
-        p.setX(qMax(0.0, qMin(value, pegi->rect().width())));
+      p.setX(qMax(qreal(0.0), qMin(qreal(value), pegi->rect().width())));
     }
     else {
-        p.setY(qMax(0.0, qMin(value, pegi->rect().height())));
+      p.setY(qMax(qreal(0.0), qMin(qreal(value), pegi->rect().height())));
     }
     
     terminalPointChangedAux(pegi, pegi->terminalPoint(), p);
Index: fritzing-0.8.3b/src/partsbinpalette/graphicsflowlayout.cpp
===================================================================
--- fritzing-0.8.3b.orig/src/partsbinpalette/graphicsflowlayout.cpp	2013-08-15 20:23:37.430572568 +0200
+++ fritzing-0.8.3b/src/partsbinpalette/graphicsflowlayout.cpp	2013-08-15 20:23:37.426572568 +0200
@@ -81,7 +81,7 @@
 
 		x = nextX;
 		// item->preferredSize().height() returns qreal, armel compiler complains
-		lineHeight = qMax(lineHeight, item->preferredSize().height());
+		lineHeight = qMax(lineHeight, (double) item->preferredSize().height());
 	}
 
 	m_lastWidth = rect.width();
