Description: Qreal fix.patch
Index: fritzing-0.9.0b/src/partseditor/pemainwindow.cpp
===================================================================
--- fritzing-0.9.0b.orig/src/partseditor/pemainwindow.cpp	2014-09-29 21:29:00.693242124 +0200
+++ fritzing-0.9.0b/src/partseditor/pemainwindow.cpp	2014-09-29 21:29:00.681242124 +0200
@@ -2420,10 +2420,10 @@
 
     QPointF p = pegi->terminalPoint();
     if (coord == "x") {
-        p.setX(qMax(0.0, qMin(value, pegi->rect().width())));
+      p.setX(qMax(qreal(0.0), qMin(qreal(value), pegi->rect().width())));
     }
     else {
-        p.setY(qMax(0.0, qMin(value, pegi->rect().height())));
+      p.setY(qMax(qreal(0.0), qMin(qreal(value), pegi->rect().height())));
     }
     
     terminalPointChangedAux(pegi, pegi->terminalPoint(), p);
Index: fritzing-0.9.0b/src/partsbinpalette/graphicsflowlayout.cpp
===================================================================
--- fritzing-0.9.0b.orig/src/partsbinpalette/graphicsflowlayout.cpp	2014-09-29 21:29:00.693242124 +0200
+++ fritzing-0.9.0b/src/partsbinpalette/graphicsflowlayout.cpp	2014-09-29 21:29:00.681242124 +0200
@@ -79,7 +79,7 @@
 
 		x = nextX;
 		// item->preferredSize().height() returns qreal, armel compiler complains
-		lineHeight = qMax(lineHeight, item->preferredSize().height());
+		lineHeight = qMax(lineHeight, (double) item->preferredSize().height());
 	}
 
 	m_lastWidth = rect.width();
