Just a typo fix.
Index: luminance-2.4.0/i18n/lang_cs.ts
===================================================================
--- luminance-2.4.0.orig/i18n/lang_cs.ts	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/i18n/lang_cs.ts	2014-01-28 18:58:16.138792046 +0100
@@ -483,7 +483,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.4.0/i18n/lang_de.ts
===================================================================
--- luminance-2.4.0.orig/i18n/lang_de.ts	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/i18n/lang_de.ts	2014-01-28 18:58:16.138792046 +0100
@@ -484,7 +484,7 @@
         <translation>Ausgabe</translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation>Breite des Ausgabebildes:</translation>
     </message>
     <message>
Index: luminance-2.4.0/i18n/lang_es.ts
===================================================================
--- luminance-2.4.0.orig/i18n/lang_es.ts	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/i18n/lang_es.ts	2014-01-28 18:58:16.142792046 +0100
@@ -483,7 +483,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.4.0/i18n/lang_it.ts
===================================================================
--- luminance-2.4.0.orig/i18n/lang_it.ts	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/i18n/lang_it.ts	2014-01-28 18:58:16.154792046 +0100
@@ -474,7 +474,7 @@
         <translation>Uscita</translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation>Larghezza Immagine:</translation>
     </message>
     <message>
Index: luminance-2.4.0/i18n/lang_pl.ts
===================================================================
--- luminance-2.4.0.orig/i18n/lang_pl.ts	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/i18n/lang_pl.ts	2014-01-28 18:58:16.154792046 +0100
@@ -435,7 +435,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.4.0/src/BatchTM/BatchTMDialog.ui
===================================================================
--- luminance-2.4.0.orig/src/BatchTM/BatchTMDialog.ui	2014-01-28 18:58:16.166792046 +0100
+++ luminance-2.4.0/src/BatchTM/BatchTMDialog.ui	2014-01-28 18:58:16.158792046 +0100
@@ -273,7 +273,7 @@
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
-               <string>Ouput Image Width:</string>
+               <string>Output Image Width:</string>
               </property>
              </widget>
             </item>
