Just a typo fix.
Index: luminance-2.3.1/i18n/lang_cs.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_cs.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_cs.ts	2013-04-12 18:03:50.216360938 +0200
@@ -443,7 +443,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_de.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_de.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_de.ts	2013-04-12 18:03:50.216360938 +0200
@@ -446,7 +446,7 @@
         <translation>Qualität des Ausgabebildes</translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation>Breite des Ausgabebildes:</translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_es.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_es.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_es.ts	2013-04-12 18:03:50.220360938 +0200
@@ -443,7 +443,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_fi.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_fi.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_fi.ts	2013-04-12 18:03:50.220360938 +0200
@@ -444,7 +444,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_hi.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_hi.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_hi.ts	2013-04-12 18:03:50.228360936 +0200
@@ -496,7 +496,7 @@
     </message>
     <message>
         <location filename="../src/BatchTM/BatchTMDialog.ui" line="297"/>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_hu.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_hu.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_hu.ts	2013-04-12 18:03:50.232360936 +0200
@@ -443,7 +443,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_id.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_id.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_id.ts	2013-04-12 18:03:50.232360936 +0200
@@ -443,7 +443,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_it.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_it.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_it.ts	2013-04-12 18:03:50.232360936 +0200
@@ -455,7 +455,7 @@
         <translation>Qualità immagine</translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation>Larghezza Immagine:</translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_pl.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_pl.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_pl.ts	2013-04-12 18:03:50.232360936 +0200
@@ -443,7 +443,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_ro.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_ro.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_ro.ts	2013-04-12 18:03:50.232360936 +0200
@@ -444,7 +444,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_ru.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_ru.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_ru.ts	2013-04-12 18:03:50.232360936 +0200
@@ -451,7 +451,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_sk.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_sk.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_sk.ts	2013-04-12 18:03:50.236360936 +0200
@@ -496,7 +496,7 @@
     </message>
     <message>
         <location filename="../src/BatchTM/BatchTMDialog.ui" line="297"/>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
Index: luminance-2.3.1/i18n/lang_zh.ts
===================================================================
--- luminance-2.3.1.orig/i18n/lang_zh.ts	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/i18n/lang_zh.ts	2013-04-12 18:03:50.236360936 +0200
@@ -519,7 +519,7 @@
     </message>
     <message>
         <location filename="../src/BatchTM/BatchTMDialog.ui" line="297"/>
-        <source>Ouput Image Width:</source>
+        <source>Output Image Width:</source>
         <translation type="unfinished">输出图片宽度:</translation>
     </message>
     <message>
Index: luminance-2.3.1/src/BatchTM/BatchTMDialog.ui
===================================================================
--- luminance-2.3.1.orig/src/BatchTM/BatchTMDialog.ui	2013-04-07 20:41:14.000000000 +0200
+++ luminance-2.3.1/src/BatchTM/BatchTMDialog.ui	2013-04-12 18:03:50.236360936 +0200
@@ -294,7 +294,7 @@
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
-               <string>Ouput Image Width:</string>
+               <string>Output Image Width:</string>
               </property>
              </widget>
             </item>
