# add missing file for translation
Index: gnome-twitch-0.1.0/po/POTFILES.in
===================================================================
--- gnome-twitch-0.1.0.orig/po/POTFILES.in	2015-10-17 18:24:38.000000000 +0200
+++ gnome-twitch-0.1.0/po/POTFILES.in	2015-10-25 03:04:17.103717661 +0100
@@ -1,4 +1,5 @@
 data/ui/gt-win.ui
 data/ui/app-menu.ui
 data/ui/gt-settings-dlg.ui
+data/ui/gt-player-header-bar.ui
 src/gt-win.c
