# revert "Adapt gettext call to new meson syntax"
Index: gnome-twitch-0.2.1/po/meson.build
===================================================================
--- gnome-twitch-0.2.1.orig/po/meson.build	2016-08-07 22:47:04.000000000 +0200
+++ gnome-twitch-0.2.1/po/meson.build	2016-08-08 15:10:25.554160123 +0200
@@ -1,5 +1,3 @@
-i18n = import('i18n')
-
 langs = ['de', 'pl', 'pt', 'sv', 'fr', 'ru', 'sr', 'hu']
 
-i18n.gettext('gnome-twitch', languages : langs)
+gettext('gnome-twitch', languages : langs)
