Source directory was missing in the includes.
Index: yagf-0.9.2/CMakeLists.txt
===================================================================
--- yagf-0.9.2.orig/CMakeLists.txt	2012-09-04 18:28:19.000000000 +0200
+++ yagf-0.9.2/CMakeLists.txt	2012-09-04 18:29:45.482852936 +0200
@@ -31,7 +31,7 @@
 qt4_wrap_ui(UI_HEADERS ${UIS})
 qt4_wrap_cpp(MOC_SRCS ${MOC_HEADERS})
 qt4_add_resources(yagf_RCC_SRCS ${yagf_RCCS})
-include_directories( ${CMAKE_BINARY_DIR} ${PROJECT_SOURCE_DIR}/src)
+include_directories( ${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR} ${PROJECT_SOURCE_DIR}/src)
 add_executable(yagf ${SOURCES} ${UI_HEADERS} ${MOC_SRCS} ${yagf_RCC_SRCS} ${QM_FILES})
 add_library(xspreload SHARED ${INT_SOURCES})
 target_link_libraries(xspreload dl)
