diff -U 3 -H -w -E -d -r -N -- AriaSrc-1.4.10.orig/Src/Dialogs/TrackPropertiesDialog.cpp AriaSrc-1.4.10/Src/Dialogs/TrackPropertiesDialog.cpp
--- AriaSrc-1.4.10.orig/Src/Dialogs/TrackPropertiesDialog.cpp	2014-10-28 18:03:14.000000000 +0000
+++ AriaSrc-1.4.10/Src/Dialogs/TrackPropertiesDialog.cpp	2014-10-28 18:15:38.238822175 +0000
@@ -37,6 +37,9 @@
 #include <wx/stattext.h>
 #include <iostream>
 
+static const int ID_CHECK_ALL_BUTTON = 10000;
+static const int ID_UNCHECK_ALL_BUTTON = 10001;
+
 namespace AriaMaestosa
 {
     class BackgroundChoicePanel : public wxPanel
@@ -179,14 +182,10 @@
                 {
                     wxPanel* checkPanel = new wxPanel(properties_panel);
                     wxBoxSizer* check_sizer = new wxBoxSizer(wxHORIZONTAL);
-                    m_checkall_btn = new wxButton(checkPanel, wxID_ANY, _("Check all"));
-                    m_uncheckall_btn = new wxButton(checkPanel, wxID_ANY, _("Uncheck all"));
+                    m_checkall_btn = new wxButton(checkPanel, ID_CHECK_ALL_BUTTON, _("Check all"));
+                    m_uncheckall_btn = new wxButton(checkPanel, ID_UNCHECK_ALL_BUTTON, _("Uncheck all"));
                     check_sizer->Add(m_checkall_btn, 1, wxEXPAND |wxALL, 5);
                     check_sizer->Add(m_uncheckall_btn, 1, wxEXPAND |wxALL, 5);
-                    
-                    m_checkall_btn->Bind(wxEVT_COMMAND_BUTTON_CLICKED, &TrackPropertiesDialog::ckeckAllButton, this);
-                    m_uncheckall_btn->Bind(wxEVT_COMMAND_BUTTON_CLICKED, &TrackPropertiesDialog::unckeckAllButton, this);
-    
                     check_sizer->Layout();
                     check_sizer->SetSizeHints(this);
                     check_sizer->Fit(checkPanel);
@@ -340,6 +339,8 @@
     BEGIN_EVENT_TABLE(TrackPropertiesDialog, wxDialog)
     EVT_BUTTON(wxID_OK, TrackPropertiesDialog::okButton)
     EVT_BUTTON(wxID_CANCEL, TrackPropertiesDialog::cancelButton)
+    EVT_BUTTON(ID_CHECK_ALL_BUTTON, TrackPropertiesDialog::ckeckAllButton)
+    EVT_BUTTON(ID_UNCHECK_ALL_BUTTON, TrackPropertiesDialog::unckeckAllButton)
     
     EVT_COMMAND_SCROLL_THUMBTRACK(300, TrackPropertiesDialog::volumeSlideChanging)
     EVT_COMMAND_SCROLL_THUMBRELEASE(300, TrackPropertiesDialog::volumeSlideChanging)
diff -U 3 -H -w -E -d -r -N -- AriaSrc-1.4.10.orig/Src/Editors/ControllerEditor.cpp AriaSrc-1.4.10/Src/Editors/ControllerEditor.cpp
--- AriaSrc-1.4.10.orig/Src/Editors/ControllerEditor.cpp	2014-10-28 18:03:14.000000000 +0000
+++ AriaSrc-1.4.10/Src/Editors/ControllerEditor.cpp	2014-10-28 18:15:38.238822175 +0000
@@ -99,7 +99,7 @@
         {
             //I18N: as in "soundbank"
             labelContents = _("bank ID");
-            m_input->SetValue("0");
+            m_input->SetValue(_("0"));
         }
         
         m_units = new wxChoice(panel, wxID_ANY);
diff -U 3 -H -w -E -d -r -N -- AriaSrc-1.4.10.orig/Src/GUI/MainFrame.cpp AriaSrc-1.4.10/Src/GUI/MainFrame.cpp
--- AriaSrc-1.4.10.orig/Src/GUI/MainFrame.cpp	2014-10-28 18:03:14.000000000 +0000
+++ AriaSrc-1.4.10/Src/GUI/MainFrame.cpp	2014-10-28 18:15:38.238822175 +0000
@@ -236,7 +236,11 @@
 
 void CustomToolBar::AddCheckTool(const int id, wxString label, wxBitmap& bmp, bool checked)
 {
+#if wxCHECK_VERSION(2,9,1)
     wxBitmapToggleButton* btn = new wxBitmapToggleButton(this, id, bmp);
+#else
+    wxToggleButton* btn = new wxToggleButton(this, id, label);
+#endif
     btn->SetValue(checked);
     toolbarSizer->Add(btn, 0, wxALIGN_CENTER | wxALIGN_CENTER_VERTICAL | wxALL, 5);
     labels.push_back(label);
@@ -786,7 +790,7 @@
     if (!loop_enabled)
     {
         m_toolbar->realize();
-        m_toolbar->SetLabelById(LOOP_END_MEASURE, "");
+        m_toolbar->SetLabelById(LOOP_END_MEASURE, _(""));
         m_toolbar->HideById(LOOP_END_MEASURE);
     }
 
@@ -1090,12 +1094,12 @@
     m_loop_end_measure->Enable(pressed);
     if (not pressed)
     {
-        m_toolbar->SetLabelById(LOOP_END_MEASURE, "");
+        m_toolbar->SetLabelById(LOOP_END_MEASURE, _(""));
         m_toolbar->HideById(LOOP_END_MEASURE);
     }
     else
     {
-        if (m_loop_end_measure->GetValue() == "1")
+        if (m_loop_end_measure->GetValue() == _("1"))
         {
             int measure_count = getCurrentSequence()->getMeasureData()->getMeasureAmount();
             m_loop_end_measure->SetValue(to_wxString(measure_count));
