# add X11 lib
Index: kchmviewer-7.3/CMakeLists.txt
===================================================================
--- kchmviewer-7.3.orig/CMakeLists.txt	2014-04-13 08:10:26.255372000 +0200
+++ kchmviewer-7.3/CMakeLists.txt	2015-02-24 21:23:10.669509268 +0100
@@ -12,6 +12,9 @@
 # libzip
 FIND_PATH(LIBZIP_INCLUDE_DIR zip.h)
 FIND_LIBRARY(LIBZIP_LIBRARY NAMES zip)
+# x11
+FIND_PATH(LIBX11_INCLUDE_DIR Xutil.h)
+FIND_LIBRARY(LIBX11_LIBRARY NAMES X11)
 
 IF (LIBZIP_INCLUDE_DIR AND LIBZIP_LIBRARY)
 	MESSAGE(STATUS "Found libzip: ${LIBZIP_LIBRARY}")
Index: kchmviewer-7.3/src/CMakeLists.txt
===================================================================
--- kchmviewer-7.3.orig/src/CMakeLists.txt	2014-04-13 08:10:26.255372000 +0200
+++ kchmviewer-7.3/src/CMakeLists.txt	2015-02-24 21:23:43.897508280 +0100
@@ -53,7 +53,8 @@
 	${LIBZIP_LIBRARY}
 	${KDE4_KDEUI_LIBS} 
 	${QT_QTWEBKIT_LIBRARY}	
-	${KDE4_KFILE_LIBS}  )
+	${KDE4_KFILE_LIBS}
+	${LIBX11_LIBRARY}  )
 
 INSTALL( TARGETS kchmviewer DESTINATION ${KDE4_BIN_INSTALL_DIR} )
 
