Description: disable check new version
Bug: 602535
Index: kchmviewer-7.1/src/config.cpp
===================================================================
--- kchmviewer-7.1.orig/src/config.cpp	2014-10-13 03:01:39.693058506 +0200
+++ kchmviewer-7.1/src/config.cpp	2014-10-13 03:01:39.689058506 +0200
@@ -49,7 +49,7 @@
 	m_advLayoutDirectionRL = settings.value( "advanced/layoutltr", false ).toBool();
 	m_advAutodetectEncoding = settings.value( "advanced/autodetectenc", false ).toBool();
 	m_advExternalEditorPath = settings.value( "advanced/editorpath", "/usr/bin/kate" ).toString();
-	m_advCheckNewVersion = settings.value( "advanced/checknewver", true ).toBool();
+	m_advCheckNewVersion = false;
 	m_toolbarMode = (Config::ToolbarMode) settings.value( "advanced/toolbarmode", TOOLBAR_LARGEICONSTEXT ).toInt();
 	m_lastOpenedDir = settings.value( "advanced/lastopendir", "." ).toString();
 
Index: kchmviewer-7.1/src/dialog_setup.cpp
===================================================================
--- kchmviewer-7.1.orig/src/dialog_setup.cpp	2014-10-13 03:01:39.693058506 +0200
+++ kchmviewer-7.1/src/dialog_setup.cpp	2014-10-13 03:01:39.689058506 +0200
@@ -172,7 +172,7 @@
 	
 	// Autodetect encoding
 	Check_Need_Restart( boxAutodetectEncoding, &pConfig->m_advAutodetectEncoding, &need_restart );
-	pConfig->m_advCheckNewVersion = cbCheckForUpdates->isChecked();
+	pConfig->m_advCheckNewVersion = false;
 
 	// Layout direction management
 	bool layout_rl = boxLayoutDirectionRL->isChecked();
